<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">

<html>
        <head>
                <meta http-equiv="content-type" content="text/html; charset=UTF-8" />

                <link href="styles/main.css" rel="stylesheet" media="screen" type="text/css" />

                <script src="scripts/jquery-3.1.1.min.js"></script>
                <script src="scripts/unslider.min.js"></script>
                <script src="scripts/autoplay.js"></script>
                
                <title>Publication</title>
	</head>


	<body>
                <div class="f_navigator1">
                        <div class="f_navigator2" style="left:0px"><a href="index.html"><div class="navigator1">首页</div></a></div>
                        <div class="f_navigator2" style="left:240px">
                                <a href="our_lab.html"><div class="navigator1">成员介绍</div></a>
                                <ul class="subnav">
                                <li><a href="our_lab.html">现有成员</a></li>
                                <li><a href="former_member.html">过去成员</a></li>
                                </ul>
                        </div>
                        <div class="f_navigator2" style="left:480px"><a href="research.html"><div class="navigator1">研究方向</div></a></div>
                        <div class="f_navigator2" style="left:720px"><a href="publication.html"><div class="navigator2">研究成果</div></a></div>
                        <!--<div class="f_navigator2" style="left:800px"><a href="project.html"><div class="navigator1">重点项目</div></a></div>-->
                        <div class="f_navigator2" style="left:960px"><a href="join_us.html"><div class="navigator1">招贤纳士</div></a></div>
                </div>

               <!-- <div class="f_img1">
                        <div class="banner">
                                <ul>
                                        <li style="list-style:none; float:left"><img src="images/b3.jpg" /></li>
                                        <li style="list-style:none; float:left"><img src="images/b2.jpg" /></li>
                                        <li style="list-style:none; float:left"><img src="images/b1.jpg" /></li>
                                        <li style="list-style:none; float:left"><img src="images/b4.jpg" /></li>
                                        <li style="list-style:none; float:left"><img src="images/b5.jpg" /></li>
                                </ul>
                        </div>
                </div>

                <div class="f_title1"><div class="title1">研究成果</div></div>-->

                 
           <div style="background-color: rgb(240, 240, 240);top:6px" id="div1">
<ul>
            <!--<li><a href="https://www.nature.com/articles/s41422-023-00849-5"><img src="../../images/cover/cover16.jpg"></li></a>-->
            <li><a href="https://academic.oup.com/mbe/article/34/3/640/2823562"><img src="images/cover/cover15.jpeg"></li></a>
            <li><a href="https://royalsocietypublishing.org/doi/full/10.1098/rspb.2018.1523"><img src="images/cover/cover14.jpg"></li></a>
            <li><a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/mec.14873"><img src="images/cover/cover13.jpg"></li></a>
            <li><a href="https://royalsocietypublishing.org/doi/full/10.1098/rsbl.2019.0461"><img src="images/cover/cover12.jpg"></li></a>
            <li><a href="https://academic.oup.com/mbe/article/36/10/2171/5509525"><img src="images/cover/cover11.jpeg"></li></a>
            <li><a href="https://link.springer.com/article/10.1186/s12862-019-1502-4"><img src="images/cover/cover10.jpg"></li></a>
            <li><a href="https://www.cell.com/current-biology/fulltext/S0960-9822(20)31337-3"><img src="images/cover/cover09.jpg"></li></a>
            <li><a href="https://www.pnas.org/doi/abs/10.1073/pnas.2021516118"><img src="images/cover/cover08.jpg"></li></a>
            <li><a href="https://www.nature.com/articles/s41559-021-01407-1"><img src="images/cover/cover07.jpg"></li></a>
            <li><a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/1749-4877.12509"><img src="images/cover/cover06.jpg"></li></a>
            <li><a href="https://academic.oup.com/mbe/article/38/10/4562/6317515"><img src="images/cover/cover05.jpg"></li></a>
            <li><a href="https://www.pnas.org/doi/abs/10.1073/pnas.2218183120"><img src="images/cover/cover04.jpg"></li></a>
            <li><a href="https://www.science.org/doi/full/10.1126/sciadv.add0141"><img src="images/cover/cover03.jpg"></li></a>
            <li><a href="https://www.sciencedirect.com/science/article/abs/pii/S2095927323004619"><img src="images/cover/cover02.jpg"></li></a>
            <li><a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/jse.13060"><img src="images/cover/cover01.jpg"></li></a>
<div style="clear: none;"></div>
</ul>
</div> 

                <div class="f_container1" style="top:320px">
                        <div class="context4">
                                <ol>
                                        

<h2>-- 研究成果 --</h2>
<h3 class="year-title">2024</h3>
<li><p><a href="https://doi.org/10.1111/jse.13060">Yue, Y., Zou, D., Tian, S., <b>Jiao, H.</b>, & Zhao, H. (2024). Comparative genomics reveals convergent adaptations in lipid metabolism and low genetic diversity in carnivorous bats. Journal of Systematics and Evolution, 62(5), 843–857. </a></p></li>
<li><p><a href="https://doi.org/10.1111/jse.13059">Xu, Y., Li, Y., Hu, H., <b>Jiao, H.</b>, & Zhao, H. (2024). Genomic and functional insights into dietary diversification in New World leaf-nosed bats (Phyllostomidae). Journal of Systematics and Evolution, 62(5), 928–941.</a></p></li>
<h3 class="year-title">2023</h3>
<li><p><a href="https://doi.org/10.1016/j.scib.2023.07.026">Li, Y., <b>Jiao, H.</b>, Sin, S. Y. W., Wang, R., Rossiter, S. J., & Zhao, H. (2023). Common ancestors of bats were omnivorous suggested by resurrection of ancestral sweet receptors. Science Bulletin, 68(16), 1748–1751. </a></p></li>
<li><p><a href="https://doi.org/10.1126/sciadv.add0141">Tian, S., Zeng, J., <b>Jiao, H.</b>, Zhang, D., Zhang, L., Lei, C., Rossiter, S. J., & Zhao, H. (2023). Comparative analyses of bat genomes identify distinct evolution of immunity in Old World fruit bats. Science Advances, 9(18), eadd0141. </a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.2218183120">Hao, X., <b>Jiao, H.</b>, Zou, D., Li, Q., Yuan, X., Liao, W., Jiang, P., & Zhao, H. (2023). Evolution of bitter receptor genes and ontogenetic dietary shift in a frog. Proceedings of the National Academy of Sciences, 120(8), e2218183120. </a></p></li>
<h3 class="year-title">2021</h3>
<li><p><a href="https://doi.org/10.1093/molbev/msab205"><b>Jiao, H.</b>, Wang, Q., Wang, B.-J., Li, K., Lövy, M., Nevo, E., Li, Q., Su, W., Jiang, P., & Zhao, H. (2021). Local Adaptation of Bitter Taste and Ecological Speciation in a Wild Mammal. Molecular Biology and Evolution, 38(10), 4562–4572. </a></p></li>
<li><p><a href="https://doi.org/10.1111/1749-4877.12509">Lu, Q., <b>Jiao, H.</b>, Wang, Y., Norbu, N., & Zhao, H. (2021). Molecular evolution and deorphanization of bitter taste receptors in a vampire bat. Integrative Zoology, 16(5), 659–669. </a></p></li>
<li><p><a href="https://doi.org/10.1038/s41559-021-01407-1">Yan, H., <b>Jiao, H.</b>, Liu, Q., Zhang, Z., Xiong, Q., Wang, B.-J., Wang, X., Guo, M., Wang, L.-F., Lan, K., Chen, Y., & Zhao, H. (2021). ACE2 receptor usage reveals variation in susceptibility to SARS-CoV and SARS-CoV-2 infection among bat species. Nature Ecology & Evolution, 5(5), 600–608. </a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.2021516118"><b>Jiao, H.</b>, Xie, H.-W., Zhang, L., Zhuoma, N., Jiang, P., & Zhao, H. (2021). Loss of sweet taste despite the conservation of sweet receptor genes in insectivorous bats. Proceedings of the National Academy of Sciences, 118(4), e2021516118. </a></p></li>
<h3 class="year-title">2020</h3>
<li><p><a href="https://doi.org/10.1016/j.cub.2020.09.006">Li, W., Huang, X., Zou, J., Wu, J., <b>Jiao, H.</b>, Peng, X., & Sun, M. (2020). Three STIGMA AND STYLE STYLISTs Pattern the Fine Architectures of Apical Gynoecium and Are Critical for Male Gametophyte-Pistil Interaction. Current Biology, 30(23), 4780-4788.e5. </a></p></li>
<li><p><a href="https://doi.org/10.1101/2020.09.08.284737">Yan, H., <b>Jiao, H.</b>, Liu, Q., Zhang, Z., Wang, X., Guo, M., Wang, B.-J., Lan, K., Chen, Y., & Zhao, H. (2020). Many bat species are not potential hosts of SARS-CoV and SARS-CoV-2: Evidence from ACE2 receptor usage. bioRxiv, 2020.09.08.284737. </a></p></li>
<li><p><a href="https://doi.org/10.1038/s41467-019-14231-w">Zhou, Z.-S., Li, M.-X., Liu, J., <b>Jiao, H.</b>, Xia, J.-M., Shi, X.-J., Zhao, H., Chu, L., Liu, J., Qi, W., Luo, J., & Song, B.-L. (2020). Competitive oxidation and ubiquitylation on the evolutionarily conserved cysteine confer tissue-specific stabilization of Insig-2. Nature Communications, 11(1), 379. </a></p></li>
<h3 class="year-title">2019</h3>
<li><p><a href="https://doi.org/10.1186/s12862-019-1502-4"><b>Jiao, H.</b>, Hong, W., Nevo, E., Li, K., & Zhao, H. (2019). Convergent reduction of V1R genes in subterranean rodents. BMC Evolutionary Biology, 19(1), 176. </a></p></li>
<li><p><a href="https://doi.org/10.1093/molbev/msz127"><b>Jiao, H.</b>, Zhang, L., Xie, H.-W., Simmons, N. B., Liu, H., & Zhao, H. (2019). Trehalase Gene as a Molecular Signature of Dietary Diversification in Mammals. Molecular Biology and Evolution, 36(10), 2171–2183. </a></p></li>
<li><p><a href="https://doi.org/10.1098/rsbl.2019.0461">Wang, Y., <b>Jiao, H.</b>, Jiang, P., & Zhao, H. (2019). Functional divergence of bitter taste receptors in a nectar-feeding bird. Biology Letters, 15(9), 20190461. </a></p></li>
<h3 class="year-title">2018</h3>
<li><p><a href="https://doi.org/10.1111/mec.14873"><b>Jiao, H.</b>, Wang, Y., Zhang, L., Jiang, P., & Zhao, H. (2018). Lineage-specific duplication and adaptive evolution of bitter taste receptor genes in bats. Molecular Ecology, 27(22), 4475–4488. </a></p></li>
<li><p><a href="https://doi.org/10.1098/rspb.2018.1523">Wu, J., <b>Jiao, H.</b>, Simmons, N. B., Lu, Q., & Zhao, H. (2018). Testing the sensory trade-off hypothesis in New World bats. Proceedings of the Royal Society B: Biological Sciences, 285(1885), 20181523. </a></p></li>
<h3 class="year-title">2017</h3>
<li><p><a href="https://doi.org/10.1038/s41598-017-01835-9">Wang, K., Hong, W., <b>Jiao, H.</b>, & Zhao, H. (2017). Transcriptome sequencing and phylogenetic analysis of four species of luminescent beetles. Scientific Reports, 7(1), 1814. </a></p></li>
<li><p><a href="https://doi.org/10.1093/molbev/msw263">Zhang, Y., Jiao, Y., <b>Jiao, H.</b>, Zhao, H., & Zhu, Y.-X. (2017). Two-Step Functional Innovation of the Stem-Cell Factors WUS/WOX5 during Plant Evolution. Molecular Biology and Evolution, 34(3), 640–653. </a></p></li>
<h3 class="year-title">2015</h3>
<li><p><a href="https://doi.org/10.1073/pnas.1514896112">Li, K., Hong, W., <b>Jiao, H.</b>, Wang, G.-D., Rodriguez, K. A., Buffenstein, R., Zhao, Y., Nevo, E., & Zhao, H. (2015). Sympatric speciation revealed by genome-wide divergence in the blind mole rat Spalax. Proceedings of the National Academy of Sciences, 112(38), 11905–11910. </a></p></li>
<li><p><a href="https://doi.org/10.3109/19401736.2013.855746"><b>Jiao, H.</b>, Ding, M., & Zhao, H. (2015). Sequence and organization of complete mitochondrial genome of the firefly, Aquatica leii (Coleoptera: Lampyridae). Mitochondrial DNA, 26(5), 775–776.</a></p></li>

<!--
<h2>-- 2024年研究成果 --</h2>
<li><p><a href="https://academic.oup.com/mbe/article/40/12/msad256/7442647">G Femerling, CV Oosterhout, S Feng, RM Bristol, <b>Guojie Zhang</b>, J Groombridge, MTP Gilbert, HE Morales. Genetic load and adaptive potential of a recovered avian species that narrowly avoided extinction. Molecular Biology and Evolution. (2023)</a></p></li>
<li><p><a href="https://www.nature.com/articles/s41586-023-06798-8">LF Kuderna, JC Ulirsch, S Rashid, M Ameen, L Sundaram, G Hickey, AJ Cox, H Gao, A Kumar, F Aguet, MJ Christmas, H Clawson, M Haeussler, MC Janiak, M Kuhlwilm, JD Orkin, T Bataillon, S Manu, A Valenzuela, J Bergman, M Rouselle, FE Silva, L Agueda, J Blanc, M Gut, Dd Vries, I Goodhead, RA Harris, M Raveendran, A Jensen, IS Chuma, JE Horvath, C Hvilsom, D Juan, P Frandsen, JG Schraiber, FRd Melo, F Bertuol, H Byrne, I Sampaio, I Farias, J Valsecchi, M Messias, MNd Silva, M Trivedi, R Rossi, T Hrbek, N Andriaholinirina, CJ Rabarivola, A Zaramody, CJ Jolly, J Phillips-Conroy, G Wilkerson, C Abee, JH Simmons, E Fernandez-Duque, S Kanthaswamy, F Shiferaw, D Wu, L Zhou, Y Shao, <b>G Zhang</b>, JD Keyyu, S Knauf, MD Le, E Lizano, S Merker, A Navarro, T Nadler, CC Khor, J Lee, P Tan, WK Lim, AC Kitchener, D Zinner, I Gut, AD Melin, K Guschanski, MH Schierup, RM Beck, I Karakikes, KC Wang, G Umapathy, C Roos, JP Boubli, A Siepel, A Kundaje, B Paten, K Lindblad-Toh, J Rogers, TM Bonet, KK-H Farh. Identification of constrained sequence elements across 239 primate genomes. Nature. (2023)</a></p></li>
<li><p><a href="https://www.nature.com/articles/s41422-023-00849-5">C Yang, Y Zhou, Y Song, D Wu, Y Zeng, L Nie, P Liu, S Zhang, G Chen, J Xu, H Zhou, L Zhou, X Qian, C Liu, S Tan, C Zhou, W Dai, M Xu, Y Qi, X Wang, L Guo, G Fan, A Wang, Y Deng, Y Zhang, J Jin, Y He, C Guo, G Guo, Q Zhou, X Xu, H Yang, J Wang, S Xu, Y Mao, X Jin, J Ruan, <b>G Zhang</b>. The complete and fully-phased diploid genome of a male Han Chinese. Cell Research. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1111/1462-2920.16509">PA O'Brien, S Tan, PR Frade, SJ Robbins, JP Engelberts, SC Bell, I Vanwonterghem, DJ Miller, NS Webster, <b>G Zhang</b>, DG Bourne. Validation of key sponge symbiont pathways using genome-centric metatranscriptomics. Environmental Microbiology. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1093/molbev/msad180">F Sánchez-Barreiro, BD Cahsan, MV Westbury, X Sun, A Margaryan, C Fontsere, MW Bruford, I-RM Russo, DC Kalthoff, T Sicheritz-Pontén, B Petersen, L Dalén, <b>G Zhang</b>, T Marquès-Bonet, MTP Gilbert, Y Moodley. Historic Sampling of a Vanishing Beast: Population Structure and Diversity in the Black Rhinoceros. Molecular Biology and Evolution msad180. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1093/molbev/msad173">X Zhuang, J Zhang, Y Shao, Y Ye, C Chen, L Zhou, Z Wang, X Luo, B Su, Y Yao, D Cooper, B Hu, L Wang, X Qi, J Lin, <b>G Zhang</b>, W Wang, N Sheng and DD Wu. Integrative omics reveals rapidly evolving regulatory sequences driving primate brain evolution. Molecular Biology and Evolution msad173. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41422-023-00849-5">C Yang, Y Zhou, Y Song, D Wu, Y Zeng, L Nie, P Liu, S Zhang, G Chen, J Xu, H Zhou, L Zhou, X Qian, C Liu, S Tan, C Zhou, W Dai, M Xu, Y Qi, X Wang, L Guo, G Fan, A Wang, Y Deng, Y Zhang, J Jin, Y He, C Guo, G Guo, Q Zhou, X Xu, H Yang, J Wang, S Xu, Y Mao, X Jin, J Ruan and <b>G Zhang</b>. The complete and fully-phased diploid genome of a male Han Chinese. Cell Research (2023)</a></p></li>
<li><p><a href="https://doi.org/10.24272/j.issn.2095-8137.2023.172">Y Guo, Y Shao, X Bi, B Zhang, H Wu, Y Zhou, M Li, L Yu, <b>G Zhang</b>, D Wu and X Qi. Harvesting the fruits of the first stage of the Primate Genome Project. Zoological Research 44, 725. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.cell.2023.02.005">C Shao, S Sun, K Liu, J Wang, S Li, Q Liu, BE Deagle, I Seim, A Biscontin, Q Wang, X Liu, S Kawaguchi, Y Liu, S Jarman, Y Wang, H-Y Wang, G Huang, J Hu, B Feng, C De Pitta, S Liu, R Wang, K Ma, Y Ying, G Sales, T Sun, X Wang, Y Zhang, Y Zhao, S Pan, X Hao, Y Wang, J Xu, B Yue, Y Sun, H Zhang, M Xu, Y Liu, X Jia, J Zhu, S Liu, J Ruan, <b>G Zhang</b>, H Yang, X Xu, J Wang, X Zhao, B Meyer and G Fan. The enormous repetitive Antarctic krill genome reveals environmental adaptations and population insights. Cell 186, 1279-1294.e1219. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41559-022-01974-x">Y Zhou, X Zhan, J Jin, L Zhou, J Bergman, X Li, MMC Rousselle, MR Belles, L Zhao, M Fang, J Chen, Q Fang, L Kuderna, T Marques-Bonet, H Kitayama, T Hayakawa, Y-G Yao, H Yang, DN Cooper, X Qi, D-D Wu, MH Schierup and <b>G Zhang</b>. Eighty million years of rapid evolution of the primate Y chromosome. Nature Ecology & Evolution 7, 1114-1130. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1126/sciadv.add3580">B Zhang, W Chen, Z Wang, W Pang, M Luo, S Wang, Y Shao, W He, Y Deng, L Zhou, J Chen, M Yang, Y Wu, L Wang, H Fernandez-Bellon, S Molloy, H Meunier, F Wanert, L Kuderna, T Marques-Bonet, C Roos, X Qi, M Li, Z Liu, MH Schierup, D Cooper, J Liu, Y Zheng, <b>G Zhang</b> and D Wu. Comparative genomics reveals the hybrid origin of a macaque group. Science Advances 9, eadd3580. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1126/sciadv.adc9507">X Bi, L Zhou, J Zhang, S Feng, M Hu, D Cooper, J Lin, J Li, D Wu and <b>G Zhang</b>. Lineage-specific accelerated sequences underlying primate evolution. Science Advances 9, eadc9507. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1126/science.abn6919">Y Shao, L Zhou, F Li, L Zhao, B-L Zhang, F Shao, J-W Chen, C-Y Chen, X Bi, X-L Zhuang, H-L Zhu, J Hu, Z Sun, X Li, D Wang, I Rivas-Gonzalez, S Wang, Y-M Wang, W Chen, G Li, H-M Lu, Y Liu, LFK Kuderna, KK-H Farh, P-F Fan, L Yu, M Li, Z-J Liu, GP Tiley, AD Yoder, C Roos, T Hayakawa, T Marques-Bonet, J Rogers, PD Stenson, DN Cooper, MH Schierup, Y-G Yao, Y-P Zhang, W Wang, X-G Qi, <b>G Zhang</b> and D-D Wu. Phylogenomic analyses provide insights into primate evolution. Science 380, 913-924. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1126/science.abn4409">I Rivas-Gonzalez, M Rousselle, F Li, L Zhou, JY Dutheil, K Munch, Y Shao, D Wu, MH Schierup and <b>G Zhang</b>. Pervasive incomplete lineage sorting illuminates speciation and selection in primates. Science 380, eabn4409. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1126/science.abn8197">H Gao, T Hamp, J Ede, JG Schraiber, J Mcrae, M Singer-Berk, Y Yang, ASD Dietrich, PP Fiziev, LFK Kuderna, L Sundaram, Y Wu, A Adhikari, Y Field, C Chen, S Batzoglou, F Aguet, G Lemire, R Reimers, D Balick, MC Janiak, M Kuhlwilm, JD Orkin, S Manu, A Valenzuela, J Bergman, M Rousselle, FE Silva, L Agueda, J Blanc, M Gut, D De Vries, I Goodhead, RA Harris, M Raveendran, A Jensen, IS Chuma, JE Horvath, C Hvilsom, D Juan, P Frandsen, FR De Melo, F Bertuol, H Byrne, I Sampaio, I Farias, JV Do Amaral, M Messias, MNF Da Silva, M Trivedi, R Rossi, T Hrbek, N Andriaholinirina, CJ Rabarivola, A Zaramody, CJ Jolly, J Phillips-Conroy, G Wilkerson, C Abee, JH Simmons, E Fernandez-Duque, S Kanthaswamy, F Shiferaw, D Wu, L Zhou, Y Shao, <b>G Zhang</b>, JD Keyyu, S Knauf, MD Le, E Lizano, S Merker, A Navarro, T Bataillon, T Nadler, CC Khor, J Lee, P Tan, WK Lim, AC Kitchener, D Zinner, I Gut, A Melin, K Guschanski, MH Schierup, RMD Beck, G Umapathy, C Roos, JP Boubli, M Lek, S Sunyaev, A Odonnell-Luria, HL Rehm, J Xu, J Rogers, T Marques-Bonet and KK-H Farh. The landscape of tolerated genetic variation in humans and primates. Science 380, eabn8153. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1126/science.abn7829">LFK Kuderna, H Gao, MC Janiak, M Kuhlwilm, JD Orkin, T Bataillon, S Manu, A Valenzuela, J Bergman, M Rousselle, FE Silva, L Agueda, J Blanc, M Gut, D De Vries, I Goodhead, RA Harris, M Raveendran, A Jensen, IS Chuma, JE Horvath, C Hvilsom, D Juan, P Frandsen, JG Schraiber, FR De Melo, F Bertuol, H Byrne, I Sampaio, I Farias, J Valsecchi, M Messias, MNF Da Silva, M Trivedi, R Rossi, T Hrbek, N Andriaholinirina, CJ Rabarivola, A Zaramody, CJ Jolly, J Phillips-Conroy, G Wilkerson, C Abee, JH Simmons, E Fernandez-Duque, S Kanthaswamy, F Shiferaw, D Wu, L Zhou, Y Shao, <b>G Zhang</b>, JD Keyyu, S Knauf, MD Le, E Lizano, S Merker, A Navarro, T Nadler, CC Khor, J Lee, P Tan, WK Lim, AC Kitchener, D Zinner, I Gut, AD Melin, K Guschanski, MH Schierup, RMD Beck, G Umapathy, C Roos, JP Boubli, J Rogers, KK-H Farh and T Marques Bonet. A global catalog of whole-genome diversity from 233 primate species. Science 380, 906-913. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1126/science.abl8621">X Qi, J Wu, L Zhao, L Wang, X Guang, PA Garber, C Opie, Y Yuan, R Diao, G Li, K Wang, R Pan, W Ji, H Sun, Z Huang, C Xu, AB Witarto, R Jia, C Zhang, C Deng, Q Qiu, <b>G Zhang</b>, CC Grueter, D Wu and B Li. Adaptations to a cold climate promoted social evolution in Asian colobine primates. Science 380, eabl8621. (2023)</a></p></li>
<li><p><a href="https://www.protocols.io/view/protocols-for-the-draft-genome-assembly-of-the-eas-q26g7bx51lwz/v1">Q Li, Q Guo, Y Zhou, H Tan, T Bertozzi, Y Zhu, J Li, S Donnellan and <b>G Zhang</b>. Protocols for the draft genome assembly of the eastern banjo frog Limnodynastes dumerilii dumerilii V.1. protocols. io (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.tig.2023.01.005">K Theissinger, C Fernandes, G Formenti, I Bista, PR Berg, C Bleidorn, A Bombarely, A Crottini, GR Gallo, JA Godoy, S Jentoft, J Malukiewicz, A Mouton, RA Oomen, S Paez, PJ Palsboll, C Pampoulie, MJ Ruiz-Lopez, S Secomandi, H Svardal, C Theofanopoulou, J De Vries, A-M Waldvogel, <b>G Zhang</b>, ED Jarvis, M Blint, C Ciofai, RM Waterhouse, CJ Mazzoni, J Hoglund, SA Aghayan, TS Alioto, I Almudi, N Alvarez, PC Alves, IR Amorim Do Rosario, A Antunes, P Arribas, P Baldrian, G Bertorelle, A Bohne, A Bonisoli-Alquati, LL Bostjancic, B Boussau, CM Breton, E Buzan, PF Campos, C Carreras, LFC Castro, LJ Chueca, F Čiampor, E Conti, R Cook-Deegan, D Croll, MV Cunha, F Delsuc, AB Dennis, D Dimitrov, R Faria, A Favre, OD Fedrigo, R Fernandez, GF Ficetola, J-F Flot, T Gabaldon, DR Agius, AM Giani, MTP Gilbert, T Grebenc, K Guschanski, R Guyot, B Hausdorf, O Hawlitschek, PD Heintzman, B Heinze, M Hiller, M Husemann, A Iannucci, I Irisarri, KS Jakobsen, P Klinga, A Kloch, CF Kratochwil, H Kusche, KKS Layton, JA Leonard, E Lerat, G Liti, T Manousaki, T Marques-Bonet, P Matos-Maravi, M Matschiner, F Maumus, AM Mc Cartney, S Meiri, J Melo-Ferreira, X Mengual, MT Monaghan, M Montagna, RW Myslajek, MT Neiber, V Nicolas, M Novo, P Ozretic, F Palero, L Parvulescu, M Pascual, OS Paulo, M Pavlek, C Pegueroles, L Pellissier, G Pesole, CR Primmer, A Riesgo, L Ruber, D Rubolini, D Salvi, O Seehausen, M Seidel, B Studer, S Theodoridis, M Thines, L Urban, A Vasemägi, A Vella, N Vella, SC Vernes, C Vernesi, DR Vieites, CW Wheat, G Worheide, Y Wurm and G Zammit. How genomics can help biodiversity conservation. Trends in Genetics 39, 545-559. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.jgg.2023.03.006">L Silva, T Mendes, L Ramos, <b>G Zhang</b> and A Antunes. Parallel evolution of fish bi-modal breathing and expansion of olfactory receptor (OR) genes: toward a universal ORs nomenclature. Journal of Genetics and Genomics (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41559-023-02055-3">RR Germain, S Feng, G Chen, GR Graves, JA Tobias, C Rahbek, F Lei, J Fjeldsa, PA Hosner, MTP Gilbert, <b>G Zhang</b> and D Nogues-Bravo. Species-specific traits mediate avian demographic responses under past climate change. Nature Ecology & Evolution 7, 862-872. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.2201945119">RR Germain, S Feng, L Buffan, CP Carmona, G Chen, GR Graves, JA Tobias, C Rahbek, F Lei, J Fjeldsa, PA Hosner, MTP Gilbert, <b>G Zhang</b> and D Nogues-Bravo. Changes in the functional diversity of modern bird species over the last million years. Proceedings of the National Academy of Sciences 120, e2201945119. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41586-023-05752-y">LA Bergeron, S Besenbacher, J Zheng, P Li, MF Bertelsen, B Quintard, JI Hoffman, Z Li, J St. Leger, C Shao, J Stiller, MTP Gilbert, MH Schierup and <b>G Zhang</b>. Evolution of the germline mutation rate across vertebrates. Nature 615, 285-291. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.celrep.2023.112112">P Zhang, Y Zhu, Q Guo, J Li, X Zhan, H Yu, N Xie, H Tan, N Lundholm, L Garcia-Cuetos, MD Martin, MA Subirats, Y-H Su, I Ruiz-Trillo, MQ Martindale, J-K Yu, MTP Gilbert, <b>G Zhang</b> and Q Li. On the origin and evolution of RNA editing in metazoans. Cell Reports 42, 112112. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41576-022-00547-9"><b>G Zhang</b>. The mutation rate as an evolving trait. Nature Reviews Genetics 24, 3-3. (2023)</a></p></li>

<h2>-- 2022年研究成果 --</h2>
<li><p><a href="https://doi.org/10.24272/j.issn.2095-8137.2022.001">DD Wu, XG Qi, L Yu, M Li, ZJ Liu, AD Yoder, C Roos, T Hayakawa, J Rogers, T Marques-Bonet, B Su, YG Yao, YP Zhang and <b>G Zhang</b>. Initiation of the Primate Genome Project. Zoological Research 43, 147-149. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.jgg.2021.06.013">Z Wang, J Zhang, X Xu, C Witt, Y Deng, G Chen, G Meng, S Feng, L Xu, T Szekely, <b>G Zhang</b> and Q Zhou. Phylogeny and sex chromosome evolution of Palaeognathae. Journal of Genetics and Genomics 49, 109-119. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1186/s12915-022-01427-8">H Toh, C Yang, G Formenti, K Raja, L Yan, A Tracey, W Chow, K Howe, LA Bergeron, <b>G Zhang</b>, B Haase, J Mountcastle, O Fedrigo, J Fogg, B Kirilenko, C Munegowda, M Hiller, A Jain, D Kihara, A Rhie, AM Phillippy, SA Swanson, P Jiang, DO Clegg, ED Jarvis, JA Thomson, R Stewart, MJP Chaisson and YV Bukhman. A haplotype-resolved genome assembly of the Nile rat facilitates exploration of the genetic basis of diabetes. BMC Biology 20, 245. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41559-022-01884-y">B Qiu, X Dai, P Li, RS Larsen, R Li, AL Price, G Ding, MJ Texada, X Zhang, D Zuo, Q Gao, W Jiang, T Wen, L Pontieri, C Guo, K Rewitz, Q Li, W Liu, JJ Boomsma and <b>G Zhang</b>. Canalized gene expression during development mediates caste differentiation in ants. Nature Ecology & Evolution 6, 1753-1765. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41592-022-01512-4">Y Mao and <b>G Zhang</b>. A complete, telomere-to-telomere human genome sequence presents new opportunities for evolutionary genomics. Nature Methods 19, 635-638. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.cub.2022.02.027">J Lin, D Duchene, C Caroe, O Smith, MM Ciucani, J Niemann, D Richmond, AD Greenwood, R Macphee, <b>G Zhang</b>, S Gopalakrishnan and MTP Gilbert. Probing the genomic limits of de-extinction in the Christmas Island rat. Current Biology 32, 1650-1656.e1653. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.3390/v14071370">Y Li, <b>G Zhang</b> and J Cui. Origin and Deep Evolution of Human Endogenous Retroviruses in Pan-Primates. Viruses 14, 7. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41597-022-01680-0">X Li, R Gao, G Chen, AL Price, DB Oksnebjerg, PA Hosner, Y Zhou, <b>G Zhang</b> and S Feng. Draft genome assemblies of four manakins. Scientific Data 9, 564. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41559-022-01784-1">Q Li, M Wang, P Zhang, Y Liu, Q Guo, Y Zhu, T Wen, X Dai, X Zhang, M Nagel, BH Dethlefsen, N Xie, J Zhao, W Jiang, L Han, L Wu, W Zhong, Z Wang, X Wei, W Dai, L Liu, X Xu, H Lu, H Yang, J Wang, JJ Boomsma, C Liu, <b>G Zhang</b> and W Liu. A single-cell transcriptomic atlas tracking the neural basis of division of labour in an ant superorganism. Nature Ecology & Evolution 6, 1191-1204. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1111/1755-0998.13561">F Li, RV Rane, V Luria, Z Xiong, J Chen, Z Li, RA Catullo, PC Griffin, M Schiffer, S Pearce, SF Lee, K Mcelroy, A Stocker, J Shirriffs, F Cockerell, C Coppin, CM Sgro, A Karger, JW Cain, JA Weber, G Santpere, MW Kirschner, AA Hoffmann, JG Oakeshott and <b>G Zhang</b>. Phylogenomic analyses of the genus Drosophila reveals genomic signals of climate adaptation. Molecular Ecology Resources 22, 1559-1581. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.2115635118">HA Lewin, S Richards, E Lieberman Aiden, ML Allende, JM Archibald, M Balint, KB Barker, B Baumgartner, K Belov, G Bertorelle, ML Blaxter, J Cai, ND Caperello, K Carlson, JC Castilla-Rubio, S-M Chaw, L Chen, AK Childers, JA Coddington, DA Conde, M Corominas, KA Crandall, AJ Crawford, F Dipalma, R Durbin, TE Ebenezer, SV Edwards, O Fedrigo, P Flicek, G Formenti, RA Gibbs, MTP Gilbert, MM Goldstein, JM Graves, HT Greely, IV Grigoriev, KJ Hackett, N Hall, D Haussler, KM Helgen, CJ Hogg, S Isobe, KS Jakobsen, A Janke, ED Jarvis, WE Johnson, SJM Jones, EK Karlsson, PJ Kersey, J-H Kim, WJ Kress, S Kuraku, MKN Lawniczak, JH Leebens-Mack, X Li, K Lindblad-Toh, X Liu, JV Lopez, T Marques-Bonet, S Mazard, JaK Mazet, CJ Mazzoni, EW Myers, RJ Oneill, S Paez, H Park, GE Robinson, C Roquet, OA Ryder, JSM Sabir, HB Shaffer, TM Shank, JS Sherkow, PS Soltis, B Tang, L Tedersoo, M Uliano-Silva, K Wang, X Wei, R Wetzer, JL Wilson, X Xu, H Yang, AD Yoder and <b>G Zhang</b>. The Earth BioGenome Project 2020: Starting the clock. Proceedings of the National Academy of Sciences 119, e2115635118. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.2115639118">MKN Lawniczak, R Durbin, P Flicek, K Lindblad-Toh, X Wei, JM Archibald, WJ Baker, K Belov, ML Blaxter, T Marques Bonet, AK Childers, JA Coddington, KA Crandall, AJ Crawford, RP Davey, F Di Palma, Q Fang, W Haerty, N Hall, KJ Hoff, K Howe, ED Jarvis, WE Johnson, RN Johnson, PJ Kersey, X Liu, JV Lopez, EW Myers, OV Pettersson, AM Phillippy, MF Poelchau, KD Pruitt, A Rhie, JC Castilla-Rubio, SK Sahu, NA Salmon, PS Soltis, D Swarbreck, F Thibaud-Nissen, S Wang, JL Wegrzyn, <b>G Zhang</b>, H Zhang, HA Lewin and S Richards. Standards recommendations for the Earth BioGenome Project. Proceedings of the National Academy of Sciences 119, e2115639118. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1186/s13059-022-02765-0">J Kim, C Lee, BJ Ko, DA Yoo, S Won, AM Phillippy, O Fedrigo, <b>G Zhang</b>, K Howe, J Wood, R Durbin, G Formenti, S Brown, L Cantin, CV Mello, S Cho, A Rhie, H Kim and ED Jarvis. False gene and chromosome losses in genome assemblies caused by GC content variation and repeats. Genome Biology 23, 204. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1126/sciadv.abo0099">Y Ji, S Feng, L Wu, Q Fang, A Bruniche-Olsen, JA Dewoody, Y Cheng, D Zhang, Y Hao, G Song, Y Qu, A Suh, <b>G Zhang</b>, SJ Hackett and F Lei. Orthologous microsatellites, transposable elements, and DNA deletions correlate with generation time and body mass in neoavian birds. Science Advances 8, eabo0099. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41586-022-05325-5">ED Jarvis, G Formenti, A Rhie, A Guarracino, C Yang, J Wood, A Tracey, F Thibaud-Nissen, MR Vollger, D Porubsky, H Cheng, M Asri, GA Logsdon, P Carnevali, MJP Chaisson, C-S Chin, S Cody, J Collins, P Ebert, M Escalona, O Fedrigo, RS Fulton, LL Fulton, S Garg, JL Gerton, J Ghurye, A Granat, RE Green, W Harvey, P Hasenfeld, A Hastie, M Haukness, EB Jaeger, M Jain, M Kirsche, M Kolmogorov, JO Korbel, S Koren, J Korlach, J Lee, D Li, T Lindsay, J Lucas, F Luo, T Marschall, MW Mitchell, J Mcdaniel, F Nie, HE Olsen, ND Olson, T Pesout, T Potapova, D Puiu, A Regier, J Ruan, SL Salzberg, AD Sanders, MC Schatz, A Schmitt, VA Schneider, S Selvaraj, K Shafin, A Shumate, NO Stitziel, C Stober, J Torrance, J Wagner, J Wang, A Wenger, C Xiao, AV Zimin, <b>G Zhang</b>, T Wang, H Li, E Garrison, D Haussler, I Hall, JM Zook, EE Eichler, AM Phillippy, B Paten, K Howe, KH Miga and C Human Pangenome Reference. Semi-automated assembly of high-quality diploid human reference genomes. Nature 611, 519-531. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.tree.2021.11.008">G Formenti, K Theissinger, C Fernandes, I Bista, A Bombarely, C Bleidorn, C Ciofi, A Crottini, JA Godoy, J Hoglund, J Malukiewicz, A Mouton, RA Oomen, S Paez, PJ Palsboll, C Pampoulie, MJ Ruiz-Lopez, H Svardal, C Theofanopoulou, J De Vries, A-M Waldvogel, <b>G Zhang</b>, CJ Mazzoni, ED Jarvis, M Balint, G Formenti, K Theissinger, C Fernandes, I Bista, A Bombarely, C Bleidorn, F Čiampor, C Ciofi, A Crottini, JA Godoy, J Hoglund, J Malukiewicz, A Mouton, RA Oomen, S Paez, P Palsboll, C Pampoulie, MJ Ruiz-Lopez, H Svardal, C Theofanopoulou, J De Vries, A-M Waldvogel, G Zhang, CJ Mazzoni, E Jarvis, M Balint, SA Aghayan, TS Alioto, I Almudi, N Alvarez, PC Alves, IR Amorim, A Antunes, P Arribas, P Baldrian, PR Berg, G Bertorelle, A Bohne, A Bonisoli-Alquati, LL Bostjancic, B Boussau, CM Breton, E Buzan, PF Campos, C Carreras, LF Castro, LJ Chueca, E Conti, R Cook-Deegan, D Croll, MV Cunha, F Delsuc, AB Dennis, D Dimitrov, R Faria, A Favre, OD Fedrigo, R Fernandez, GF Ficetola, J-F Flot, T Gabaldon, DR Galea Agius, GR Gallo, AM Giani, MTP Gilbert, T Grebenc, K Guschanski, R Guyot, B Hausdorf, O Hawlitschek, PD Heintzman, B Heinze, M Hiller, M Husemann, A Iannucci, I Irisarri, KS Jakobsen, S Jentoft, P Klinga, A Kloch, CF Kratochwil, H Kusche, KKS Layton, JA Leonard, E Lerat, G Liti, T Manousaki, T Marques-Bonet, P Matos-Maravi, M Matschiner, F Maumus, AM Mc Cartney, S Meiri, J Melo-Ferreira, X Mengual, MT Monaghan, M Montagna, RW Myslajek, MT Neiber, V Nicolas, M Novo, P Ozretic, F Palero, L Parvulescu, M Pascual, OS Paulo, M Pavlek, C Pegueroles, L Pellissier, G Pesole, CR Primmer, A Riesgo, L Ruber, D Rubolini, D Salvi, O Seehausen, M Seidel, S Secomandi, B Studer, S Theodoridis, M Thines, L Urban, A Vasemägi, A Vella, N Vella, SC Vernes, C Vernesi, DR Vieites, RM Waterhouse, CW Wheat, G Worheide, Y Wurm and G Zammit. The era of reference genomes in conservation genomics. Trends in Ecology & Evolution 37, 197-202. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1093/molbev/msab354">SR Fiddaman, M Vinkler, SG Spiro, H Levy, CA Emerling, AC Boyd, EA Dimopoulos, JA Vianna, TL Cole, H Pan, M Fang, <b>G Zhang</b>, T Hart, LaF Frantz and AL Smith. Adaptation and Cryptic Pseudogenization in Penguin Toll-Like Receptors. Molecular Biology and Evolution 39, msab354. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1101/gr.276395.121">L Ferrandez-Peral, X Zhan, M Alvarez-Estape, C Chiva, P Esteller-Cucala, R Garcia-Perez, E Julia, E Lizano, O Fornas, E Sabido, Q Li, T Marques-Bonet, D Juan and <b>G Zhang</b>. Transcriptome innovations in primates revealed by single-molecule long-read sequencing. Genome research 32, 1448-1462. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.cell.2022.03.034">S Feng, M Bai, I Rivas-Gonzalez, C Li, S Liu, Y Tong, H Yang, G Chen, D Xie, KE Sears, LM Franco, JD Gaitan-Espitia, RF Nespolo, WE Johnson, H Yang, PA Brandies, CJ Hogg, K Belov, MB Renfree, KM Helgen, JJ Boomsma, MH Schierup and <b>G Zhang</b>. Incomplete lineage sorting and phenotypic evolution in marsupials. Cell 185, 1646-1660.e1618. (2022)</a></p></li>
<li><p><a href="https://www.frontiersin.org/articles/10.3389/fphys.2022.852357">G Ding, Q Gao, J Chen, J Zhao, <b>G Zhang</b> and W Liu. Validation of Potential Reference Genes for Real-Time qPCR Analysis in Pharaoh Ant, Monomorium pharaonis (Hymenoptera: Formicidae). Frontiers in Physiology 13, (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.2109326119">B Demarchi, J Stiller, A Grealy, M Mackie, Y Deng, T Gilbert, J Clarke, LJ Legendre, R Boano, T Sicheritz-Ponten, J Magee, <b>G Zhang</b>, M Bunce, MJ Collins and G Miller. Ancient proteins resolve controversy over the identity of Genyornis eggshell. Proceedings of the National Academy of Sciences 119, e2109326119. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41467-022-31508-9">TL Cole, C Zhou, M Fang, H Pan, DT Ksepka, SR Fiddaman, CA Emerling, DB Thomas, X Bi, Q Fang, MR Ellegaard, S Feng, AL Smith, TA Heath, AJD Tennyson, PG Borboroglu, JR Wood, PW Hadden, S Grosser, C-A Bost, Y Cherel, T Mattern, T Hart, M-HS Sinding, LD Shepherd, RA Phillips, P Quillfeldt, JF Masello, JL Bouzat, PG Ryan, DR Thompson, U Ellenberg, P Dann, G Miller, P Dee Boersma, R Zhao, MTP Gilbert, H Yang, D-X Zhang and <b>G Zhang</b>. Genomic insights into the secondary aquatic transition of penguins. Nature Communications 13, 3912. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1111/ibi.13023">EL Cavill, S Gopalakrishnan, LC Puetz, AM Ribeiro, SST Mak, RR Da Fonseca, G Pacheco, B Dunlop, W Accouche, N Shah, A Zora, L Calabrese, M Genner, G Jones, C Guo, <b>G Zhang</b> and MTP Gilbert. Conservation genomics of the endangered Seychelles Magpie-Robin (Copsychus sechellarum): a unique insight into the history of a precious endemic bird. IBIS 164, 396-410. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.2115636118">M Blaxter, JM Archibald, AK Childers, JA Coddington, KA Crandall, F Di Palma, R Durbin, SV Edwards, JaM Graves, KJ Hackett, N Hall, ED Jarvis, RN Johnson, EK Karlsson, WJ Kress, S Kuraku, MKN Lawniczak, K Lindblad-Toh, JV Lopez, NA Moran, GE Robinson, OA Ryder, B Shapiro, PS Soltis, T Warnow, <b>G Zhang</b> and HA Lewin. Why sequence all eukaryotes? Proceedings of the National Academy of Sciences 119, e2115636118. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.7554/eLife.73577">LA Bergeron, S Besenbacher, T Turner, CJ Versoza, RJ Wang, AL Price, E Armstrong, M Riera, J Carlson, H-Y Chen, MW Hahn, K Harris, AS Kleppe, EH Lopez-Nandam, P Moorjani, SP Pfeifer, GP Tiley, AD Yoder, <b>G Zhang</b> and MH Schierup. The Mutationathon highlights the importance of reaching standardization in estimates of pedigree-based germline mutation rates. eLife 11, e73577. (2022)</a></p></li>

                        -->
                        </ol>
                        </div>
                </div>
				<!--
                <div class="f_bottom1" style="top:23900px"><div class="bottom1">© Copyright 2019 by Centre for Biodiversity Genomics, All Rights Reserved</div></div>
				-->
                <div class="f_bottom1" style="top:2192px"><div class="bottom1">©华中师范大学生命科学学院焦恒武实验室&nbsp; 地址:武汉市珞喻路152号生命科学学院&nbsp;&nbsp; 邮政编码:430072</div></div>
				
        </body>
</html>
