<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">

<html>
        <head>
                <meta http-equiv="content-type" content="text/html; charset=UTF-8" />

                <link href="styles/main.css" rel="stylesheet" media="screen" type="text/css" />

                <script src="scripts/jquery-3.1.1.min.js"></script>
                <script src="scripts/unslider.min.js"></script>
                <script src="scripts/autoplay.js"></script>
                
                <title>Publication</title>
	</head>


	<body>
                <div class="f_navigator1">
                        <div class="f_navigator2" style="left:0px"><a href="index.html"><div class="navigator1">首页</div></a></div>
                        <div class="f_navigator2" style="left:240px"><a href="our_lab.html"><div class="navigator1">成员介绍</div></a></div>
                        <div class="f_navigator2" style="left:480px"><a href="research.html"><div class="navigator1">研究方向</div></a></div>
                        <div class="f_navigator2" style="left:720px"><a href="publication.html"><div class="navigator2">研究成果</div></a></div>
                        <!--<div class="f_navigator2" style="left:800px"><a href="project.html"><div class="navigator1">重点项目</div></a></div>-->
                        <div class="f_navigator2" style="left:960px"><a href="join_us.html"><div class="navigator1">招贤纳士</div></a></div>
                </div>

                <div class="f_img1">
                        <div class="banner">
                                <ul>
                                        <li style="list-style:none; float:left"><img src="images/b3.jpg" /></li>
                                        <li style="list-style:none; float:left"><img src="images/b2.jpg" /></li>
                                        <li style="list-style:none; float:left"><img src="images/b1.jpg" /></li>
                                        <li style="list-style:none; float:left"><img src="images/b4.jpg" /></li>
                                        <li style="list-style:none; float:left"><img src="images/b5.jpg" /></li>
                                </ul>
                        </div>
                </div>

                <div class="f_title1"><div class="title1">研究成果</div></div>

                 
           <div style="top:430px;background-color: rgb(240, 240, 240);" id="div1">
<ul>
            <!--<li><a href="https://www.nature.com/articles/s41422-023-00849-5"><img src="../../images/cover/cover16.jpg"></li></a>-->
            <li><a href="https://academic.oup.com/mbe/article/34/3/640/2823562"><img src="images/cover/cover15.jpeg"></li></a>
            <li><a href="https://royalsocietypublishing.org/doi/full/10.1098/rspb.2018.1523"><img src="images/cover/cover14.jpg"></li></a>
            <li><a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/mec.14873"><img src="images/cover/cover13.jpg"></li></a>
            <li><a href="https://royalsocietypublishing.org/doi/full/10.1098/rsbl.2019.0461"><img src="images/cover/cover12.jpg"></li></a>
            <li><a href="https://academic.oup.com/mbe/article/36/10/2171/5509525"><img src="images/cover/cover11.jpeg"></li></a>
            <li><a href="https://link.springer.com/article/10.1186/s12862-019-1502-4"><img src="images/cover/cover10.jpg"></li></a>
            <li><a href="https://www.cell.com/current-biology/fulltext/S0960-9822(20)31337-3"><img src="images/cover/cover09.jpg"></li></a>
            <li><a href="https://www.pnas.org/doi/abs/10.1073/pnas.2021516118"><img src="images/cover/cover08.jpg"></li></a>
            <li><a href="https://www.nature.com/articles/s41559-021-01407-1"><img src="images/cover/cover07.jpg"></li></a>
            <li><a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/1749-4877.12509"><img src="images/cover/cover06.jpg"></li></a>
            <li><a href="https://academic.oup.com/mbe/article/38/10/4562/6317515"><img src="images/cover/cover05.jpg"></li></a>
            <li><a href="https://www.pnas.org/doi/abs/10.1073/pnas.2218183120"><img src="images/cover/cover04.jpg"></li></a>
            <li><a href="https://www.science.org/doi/full/10.1126/sciadv.add0141"><img src="images/cover/cover03.jpg"></li></a>
            <li><a href="https://www.sciencedirect.com/science/article/abs/pii/S2095927323004619"><img src="images/cover/cover02.jpg"></li></a>
            <li><a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/jse.13060"><img src="images/cover/cover01.jpg"></li></a>
<div style="clear: none;"></div>
</ul>
</div> 

                <div class="f_container1" style="top:770px">
                        <div class="context4">
                                <ul>
                                        
<h2>-- 研究成果 --</h2>
<li><p><a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/jse.13060">Y Yue, D Zou, S Tian, <b>H Jiao</b>, H Zhao - Journal of Systematics and Evolution- -Comparative genomics reveals convergent adaptations in lipid metabolism and low genetic diversity in carnivorous bats. (2024)</a></p></li>
<li><p><a href="https://onlinelibrary.wiley.com/doi/full/10.1111/jse.13059">Y Xu, Y Li, H Hu, <b>H Jiao</b>, H Zhao - Journal of Systematics and Evolution- -Genomic and functional insights into dietary diversification in New World leaf‐nosed bats (Phyllostomidae). (2024)</a></p></li>
<li><p><a href="https://www.sciencedirect.com/science/article/abs/pii/S2095927323004619">Y Li, <b>H Jiao</b>, SYW Sin, R Wang, SJ Rossiter, H Zhao - Science Bulletin- -Common ancestors of bats were omnivorous suggested by resurrection of ancestral sweet receptors. (2023)</a></p></li>
<li><p><a href="https://www.science.org/doi/full/10.1126/sciadv.add0141">Shilin Tian, Jiaming Zeng, <b>Hengwu Jiao</b>, Dejing Zhang, Libiao Zhang, Cao-qi Lei, Stephen J Rossiter, Huabin Zhao - Science Advances- -Comparative analyses of bat genomes identify distinct evolution of immunity in Old World fruit bats. (2023)</a></p></li>
<li><p><a href="https://www.pnas.org/doi/abs/10.1073/pnas.2218183120">Xiangyu Hao, <b>Hengwu Jiao</b>, Dahu Zou, Qiyang Li, Xiangqun Yuan, Wenbo Liao, Peihua Jiang, Huabin Zhao - Proceedings of the National Academy of Sciences- -Evolution of bitter receptor genes and ontogenetic dietary shift in a frog. (2023)</a></p></li>
<li><p><a href="https://academic.oup.com/mbe/article/38/10/4562/6317515"><b>Hengwu Jiao</b>, Qian Wang, Bing-Jun Wang, Kexin Li, Matěj Lövy, Eviatar Nevo, Qiyang Li, Wenchuan Su, Peihua Jiang, Huabin Zhao - Molecular Biology and Evolution- -Local Adaptation of Bitter Taste and Ecological Speciation in a Wild Mammal. (2021)</a></p></li>
<li><p><a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/1749-4877.12509">Q Lu, <b>H Jiao</b>, Y Wang, N Norbu, H Zhao - Integrative Zoology- -Molecular evolution and deorphanization of bitter taste receptors in a vampire bat.  (2021)</a></p></li>
<li><p><a href="https://www.nature.com/articles/s41559-021-01407-1">Huan Yan, <b>Hengwu Jiao</b>, Qianyun Liu, Zhen Zhang, Qing Xiong, Bing-Jun Wang, Xin Wang, Ming Guo, Lin-Fa Wang, Ke Lan, Yu Chen ,Huabin Zhao - Nature Ecology & Evolution- -ACE2 receptor usage reveals variation in susceptibility to SARS-CoV and SARS-CoV-2 infection among bat species. (2021)</a></p></li>
<li><p><a href="https://www.pnas.org/doi/abs/10.1073/pnas.2021516118"><b>H Jiao</b>, HW Xie, L Zhang, N Zhuoma, P Jiang, H Zhao - Proceedings of the National Academy of Sciences- -Loss of sweet taste despite the conservation of sweet receptor genes in insectivorous bats. (2021)</a></p></li>
<li><p><a href="https://www.cell.com/current-biology/fulltext/S0960-9822(20)31337-3">W Li, X Huang, J Zou, J Wu, <b>H Jiao</b>, X Peng, M Sun - Current Biology- - Three STIGMA AND STYLE STYLISTs pattern the fine architectures of apical gynoecium and are critical for male gametophyte-pistil interaction. (2020)</a></p></li>
<li><p><a href="https://www.biorxiv.org/content/10.1101/2020.09.08.284737v3.abstract">Huan Yan, <b>Hengwu Jiao</b>, Qianyun Liu, Zhen Zhang, Xin Wang, Ming Guo, Bing-Jun Wang, Ke Lan, Yu Chen, Huabin Zhao - BioRxiv- -Many bat species are not potential hosts of SARS-CoV and SARS-CoV-2: Evidence from ACE2 receptor usage. (2020)</a></p></li>
<li><p><a href="https://www.nature.com/articles/s41467-019-14231-w">Zhang-Sen Zhou, Mei-Xin Li, Jie Liu, <b>Hengwu Jiao</b>, Jing-Ming Xia, Xiong-Jie Shi, Huabin Zhao, Liping Chu, Jingrong Liu, Wei Qi, Jie Luo, Bao-Liang Song - Nature Communications- -Competitive oxidation and ubiquitylation on the evolutionarily conserved cysteine confer tissue-specific stabilization of Insig-2. (2020)</a></p></li>
<li><p><a href="https://link.springer.com/article/10.1186/s12862-019-1502-4"><b>H Jiao</b>, W Hong, E Nevo, K Li, H Zhao - BMC Evolutionary Biology- -Convergent reduction of V1R genes in subterranean rodents. (2019)</a></p></li>
<li><p><a href="https://academic.oup.com/mbe/article/36/10/2171/5509525"><b>H Jiao</b>, L Zhang, HW Xie, NB Simmons, H Liu, H Zhao - Molecular Biology and Evolution- -Trehalase Gene as a Molecular Signature of Dietary Diversification in Mammals. (2019)</a></p></li>
<li><p><a href="https://royalsocietypublishing.org/doi/full/10.1098/rsbl.2019.0461">Y Wang, <b>H Jiao</b>, P Jiang, H Zhao - Biology Letters- -Functional divergence of bitter taste receptors in a nectar-feeding bird. (2019)</a></p></li>
<li><p><a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/mec.14873"><b>H Jiao</b>, Y Wang, L Zhang, P Jiang, H Zhao - Molecular ecology- -Lineage-specific duplication and adaptive evolution of bitter taste receptor genes in bats. (2018)</a></p></li>
<li><p><a href="https://royalsocietypublishing.org/doi/full/10.1098/rspb.2018.1523">J Wu, <b>H Jiao</b>, NB Simmons, Q Lu, H Zhao - Proceedings of the Royal Society B- -Testing the sensory trade-off hypothesis in New World bats. (2018)</a></p></li>
<li><p><a href="https://www.nature.com/articles/s41598-017-01835-9">K Wang, W Hong, <b>H Jiao</b>, H Zhao - Scientific Reports- -Transcriptome sequencing and phylogenetic analysis of four species of luminescent beetles. (2017)</a></p></li>
<li><p><a href="https://academic.oup.com/mbe/article/34/3/640/2823562">Y Zhang, Y Jiao, <b>H Jiao</b>, H Zhao, YX Zhu - Molecular biology and evolution- -Two-step functional innovation of the stem-cell factors WUS/WOX5 during plant evolution. (2017)</a></p></li>
<li><p><a href="https://www.pnas.org/doi/abs/10.1073/pnas.1514896112">Kexin Li, Wei Hong, <b>Hengwu Jiao</b>, Guo-Dong Wang, Karl A Rodriguez, Rochelle Buffenstein, Yang Zhao, Eviatar Nevo, Huabin Zhao - Proceedings of the National Academy of Sciences- -Sympatric speciation revealed by genome-wide divergence in the blind mole rat Spalax. (2015)</a></p></li>
<li><p><a href="https://www.tandfonline.com/doi/abs/10.3109/19401736.2013.855746"><b>H Jiao</b>, M Ding, H Zhao - Mitochondrial DNA- -Sequence and organization of complete mitochondrial genome of the firefly, Aquatica leii (Coleoptera: Lampyridae). (2015)</a></p></li>

<!--
<h2>-- 2023年研究成果 --</h2>
<li><p><a href="https://academic.oup.com/mbe/article/40/12/msad256/7442647">G Femerling, CV Oosterhout, S Feng, RM Bristol, <b>Guojie Zhang</b>, J Groombridge, MTP Gilbert, HE Morales. Genetic load and adaptive potential of a recovered avian species that narrowly avoided extinction. Molecular Biology and Evolution. (2023)</a></p></li>
<li><p><a href="https://www.nature.com/articles/s41586-023-06798-8">LF Kuderna, JC Ulirsch, S Rashid, M Ameen, L Sundaram, G Hickey, AJ Cox, H Gao, A Kumar, F Aguet, MJ Christmas, H Clawson, M Haeussler, MC Janiak, M Kuhlwilm, JD Orkin, T Bataillon, S Manu, A Valenzuela, J Bergman, M Rouselle, FE Silva, L Agueda, J Blanc, M Gut, Dd Vries, I Goodhead, RA Harris, M Raveendran, A Jensen, IS Chuma, JE Horvath, C Hvilsom, D Juan, P Frandsen, JG Schraiber, FRd Melo, F Bertuol, H Byrne, I Sampaio, I Farias, J Valsecchi, M Messias, MNd Silva, M Trivedi, R Rossi, T Hrbek, N Andriaholinirina, CJ Rabarivola, A Zaramody, CJ Jolly, J Phillips-Conroy, G Wilkerson, C Abee, JH Simmons, E Fernandez-Duque, S Kanthaswamy, F Shiferaw, D Wu, L Zhou, Y Shao, <b>G Zhang</b>, JD Keyyu, S Knauf, MD Le, E Lizano, S Merker, A Navarro, T Nadler, CC Khor, J Lee, P Tan, WK Lim, AC Kitchener, D Zinner, I Gut, AD Melin, K Guschanski, MH Schierup, RM Beck, I Karakikes, KC Wang, G Umapathy, C Roos, JP Boubli, A Siepel, A Kundaje, B Paten, K Lindblad-Toh, J Rogers, TM Bonet, KK-H Farh. Identification of constrained sequence elements across 239 primate genomes. Nature. (2023)</a></p></li>
<li><p><a href="https://www.nature.com/articles/s41422-023-00849-5">C Yang, Y Zhou, Y Song, D Wu, Y Zeng, L Nie, P Liu, S Zhang, G Chen, J Xu, H Zhou, L Zhou, X Qian, C Liu, S Tan, C Zhou, W Dai, M Xu, Y Qi, X Wang, L Guo, G Fan, A Wang, Y Deng, Y Zhang, J Jin, Y He, C Guo, G Guo, Q Zhou, X Xu, H Yang, J Wang, S Xu, Y Mao, X Jin, J Ruan, <b>G Zhang</b>. The complete and fully-phased diploid genome of a male Han Chinese. Cell Research. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1111/1462-2920.16509">PA O'Brien, S Tan, PR Frade, SJ Robbins, JP Engelberts, SC Bell, I Vanwonterghem, DJ Miller, NS Webster, <b>G Zhang</b>, DG Bourne. Validation of key sponge symbiont pathways using genome-centric metatranscriptomics. Environmental Microbiology. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1093/molbev/msad180">F Sánchez-Barreiro, BD Cahsan, MV Westbury, X Sun, A Margaryan, C Fontsere, MW Bruford, I-RM Russo, DC Kalthoff, T Sicheritz-Pontén, B Petersen, L Dalén, <b>G Zhang</b>, T Marquès-Bonet, MTP Gilbert, Y Moodley. Historic Sampling of a Vanishing Beast: Population Structure and Diversity in the Black Rhinoceros. Molecular Biology and Evolution msad180. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1093/molbev/msad173">X Zhuang, J Zhang, Y Shao, Y Ye, C Chen, L Zhou, Z Wang, X Luo, B Su, Y Yao, D Cooper, B Hu, L Wang, X Qi, J Lin, <b>G Zhang</b>, W Wang, N Sheng and DD Wu. Integrative omics reveals rapidly evolving regulatory sequences driving primate brain evolution. Molecular Biology and Evolution msad173. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41422-023-00849-5">C Yang, Y Zhou, Y Song, D Wu, Y Zeng, L Nie, P Liu, S Zhang, G Chen, J Xu, H Zhou, L Zhou, X Qian, C Liu, S Tan, C Zhou, W Dai, M Xu, Y Qi, X Wang, L Guo, G Fan, A Wang, Y Deng, Y Zhang, J Jin, Y He, C Guo, G Guo, Q Zhou, X Xu, H Yang, J Wang, S Xu, Y Mao, X Jin, J Ruan and <b>G Zhang</b>. The complete and fully-phased diploid genome of a male Han Chinese. Cell Research (2023)</a></p></li>
<li><p><a href="https://doi.org/10.24272/j.issn.2095-8137.2023.172">Y Guo, Y Shao, X Bi, B Zhang, H Wu, Y Zhou, M Li, L Yu, <b>G Zhang</b>, D Wu and X Qi. Harvesting the fruits of the first stage of the Primate Genome Project. Zoological Research 44, 725. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.cell.2023.02.005">C Shao, S Sun, K Liu, J Wang, S Li, Q Liu, BE Deagle, I Seim, A Biscontin, Q Wang, X Liu, S Kawaguchi, Y Liu, S Jarman, Y Wang, H-Y Wang, G Huang, J Hu, B Feng, C De Pitta, S Liu, R Wang, K Ma, Y Ying, G Sales, T Sun, X Wang, Y Zhang, Y Zhao, S Pan, X Hao, Y Wang, J Xu, B Yue, Y Sun, H Zhang, M Xu, Y Liu, X Jia, J Zhu, S Liu, J Ruan, <b>G Zhang</b>, H Yang, X Xu, J Wang, X Zhao, B Meyer and G Fan. The enormous repetitive Antarctic krill genome reveals environmental adaptations and population insights. Cell 186, 1279-1294.e1219. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41559-022-01974-x">Y Zhou, X Zhan, J Jin, L Zhou, J Bergman, X Li, MMC Rousselle, MR Belles, L Zhao, M Fang, J Chen, Q Fang, L Kuderna, T Marques-Bonet, H Kitayama, T Hayakawa, Y-G Yao, H Yang, DN Cooper, X Qi, D-D Wu, MH Schierup and <b>G Zhang</b>. Eighty million years of rapid evolution of the primate Y chromosome. Nature Ecology & Evolution 7, 1114-1130. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1126/sciadv.add3580">B Zhang, W Chen, Z Wang, W Pang, M Luo, S Wang, Y Shao, W He, Y Deng, L Zhou, J Chen, M Yang, Y Wu, L Wang, H Fernandez-Bellon, S Molloy, H Meunier, F Wanert, L Kuderna, T Marques-Bonet, C Roos, X Qi, M Li, Z Liu, MH Schierup, D Cooper, J Liu, Y Zheng, <b>G Zhang</b> and D Wu. Comparative genomics reveals the hybrid origin of a macaque group. Science Advances 9, eadd3580. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1126/sciadv.adc9507">X Bi, L Zhou, J Zhang, S Feng, M Hu, D Cooper, J Lin, J Li, D Wu and <b>G Zhang</b>. Lineage-specific accelerated sequences underlying primate evolution. Science Advances 9, eadc9507. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1126/science.abn6919">Y Shao, L Zhou, F Li, L Zhao, B-L Zhang, F Shao, J-W Chen, C-Y Chen, X Bi, X-L Zhuang, H-L Zhu, J Hu, Z Sun, X Li, D Wang, I Rivas-Gonzalez, S Wang, Y-M Wang, W Chen, G Li, H-M Lu, Y Liu, LFK Kuderna, KK-H Farh, P-F Fan, L Yu, M Li, Z-J Liu, GP Tiley, AD Yoder, C Roos, T Hayakawa, T Marques-Bonet, J Rogers, PD Stenson, DN Cooper, MH Schierup, Y-G Yao, Y-P Zhang, W Wang, X-G Qi, <b>G Zhang</b> and D-D Wu. Phylogenomic analyses provide insights into primate evolution. Science 380, 913-924. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1126/science.abn4409">I Rivas-Gonzalez, M Rousselle, F Li, L Zhou, JY Dutheil, K Munch, Y Shao, D Wu, MH Schierup and <b>G Zhang</b>. Pervasive incomplete lineage sorting illuminates speciation and selection in primates. Science 380, eabn4409. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1126/science.abn8197">H Gao, T Hamp, J Ede, JG Schraiber, J Mcrae, M Singer-Berk, Y Yang, ASD Dietrich, PP Fiziev, LFK Kuderna, L Sundaram, Y Wu, A Adhikari, Y Field, C Chen, S Batzoglou, F Aguet, G Lemire, R Reimers, D Balick, MC Janiak, M Kuhlwilm, JD Orkin, S Manu, A Valenzuela, J Bergman, M Rousselle, FE Silva, L Agueda, J Blanc, M Gut, D De Vries, I Goodhead, RA Harris, M Raveendran, A Jensen, IS Chuma, JE Horvath, C Hvilsom, D Juan, P Frandsen, FR De Melo, F Bertuol, H Byrne, I Sampaio, I Farias, JV Do Amaral, M Messias, MNF Da Silva, M Trivedi, R Rossi, T Hrbek, N Andriaholinirina, CJ Rabarivola, A Zaramody, CJ Jolly, J Phillips-Conroy, G Wilkerson, C Abee, JH Simmons, E Fernandez-Duque, S Kanthaswamy, F Shiferaw, D Wu, L Zhou, Y Shao, <b>G Zhang</b>, JD Keyyu, S Knauf, MD Le, E Lizano, S Merker, A Navarro, T Bataillon, T Nadler, CC Khor, J Lee, P Tan, WK Lim, AC Kitchener, D Zinner, I Gut, A Melin, K Guschanski, MH Schierup, RMD Beck, G Umapathy, C Roos, JP Boubli, M Lek, S Sunyaev, A Odonnell-Luria, HL Rehm, J Xu, J Rogers, T Marques-Bonet and KK-H Farh. The landscape of tolerated genetic variation in humans and primates. Science 380, eabn8153. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1126/science.abn7829">LFK Kuderna, H Gao, MC Janiak, M Kuhlwilm, JD Orkin, T Bataillon, S Manu, A Valenzuela, J Bergman, M Rousselle, FE Silva, L Agueda, J Blanc, M Gut, D De Vries, I Goodhead, RA Harris, M Raveendran, A Jensen, IS Chuma, JE Horvath, C Hvilsom, D Juan, P Frandsen, JG Schraiber, FR De Melo, F Bertuol, H Byrne, I Sampaio, I Farias, J Valsecchi, M Messias, MNF Da Silva, M Trivedi, R Rossi, T Hrbek, N Andriaholinirina, CJ Rabarivola, A Zaramody, CJ Jolly, J Phillips-Conroy, G Wilkerson, C Abee, JH Simmons, E Fernandez-Duque, S Kanthaswamy, F Shiferaw, D Wu, L Zhou, Y Shao, <b>G Zhang</b>, JD Keyyu, S Knauf, MD Le, E Lizano, S Merker, A Navarro, T Nadler, CC Khor, J Lee, P Tan, WK Lim, AC Kitchener, D Zinner, I Gut, AD Melin, K Guschanski, MH Schierup, RMD Beck, G Umapathy, C Roos, JP Boubli, J Rogers, KK-H Farh and T Marques Bonet. A global catalog of whole-genome diversity from 233 primate species. Science 380, 906-913. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1126/science.abl8621">X Qi, J Wu, L Zhao, L Wang, X Guang, PA Garber, C Opie, Y Yuan, R Diao, G Li, K Wang, R Pan, W Ji, H Sun, Z Huang, C Xu, AB Witarto, R Jia, C Zhang, C Deng, Q Qiu, <b>G Zhang</b>, CC Grueter, D Wu and B Li. Adaptations to a cold climate promoted social evolution in Asian colobine primates. Science 380, eabl8621. (2023)</a></p></li>
<li><p><a href="https://www.protocols.io/view/protocols-for-the-draft-genome-assembly-of-the-eas-q26g7bx51lwz/v1">Q Li, Q Guo, Y Zhou, H Tan, T Bertozzi, Y Zhu, J Li, S Donnellan and <b>G Zhang</b>. Protocols for the draft genome assembly of the eastern banjo frog Limnodynastes dumerilii dumerilii V.1. protocols. io (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.tig.2023.01.005">K Theissinger, C Fernandes, G Formenti, I Bista, PR Berg, C Bleidorn, A Bombarely, A Crottini, GR Gallo, JA Godoy, S Jentoft, J Malukiewicz, A Mouton, RA Oomen, S Paez, PJ Palsboll, C Pampoulie, MJ Ruiz-Lopez, S Secomandi, H Svardal, C Theofanopoulou, J De Vries, A-M Waldvogel, <b>G Zhang</b>, ED Jarvis, M Blint, C Ciofai, RM Waterhouse, CJ Mazzoni, J Hoglund, SA Aghayan, TS Alioto, I Almudi, N Alvarez, PC Alves, IR Amorim Do Rosario, A Antunes, P Arribas, P Baldrian, G Bertorelle, A Bohne, A Bonisoli-Alquati, LL Bostjancic, B Boussau, CM Breton, E Buzan, PF Campos, C Carreras, LFC Castro, LJ Chueca, F Čiampor, E Conti, R Cook-Deegan, D Croll, MV Cunha, F Delsuc, AB Dennis, D Dimitrov, R Faria, A Favre, OD Fedrigo, R Fernandez, GF Ficetola, J-F Flot, T Gabaldon, DR Agius, AM Giani, MTP Gilbert, T Grebenc, K Guschanski, R Guyot, B Hausdorf, O Hawlitschek, PD Heintzman, B Heinze, M Hiller, M Husemann, A Iannucci, I Irisarri, KS Jakobsen, P Klinga, A Kloch, CF Kratochwil, H Kusche, KKS Layton, JA Leonard, E Lerat, G Liti, T Manousaki, T Marques-Bonet, P Matos-Maravi, M Matschiner, F Maumus, AM Mc Cartney, S Meiri, J Melo-Ferreira, X Mengual, MT Monaghan, M Montagna, RW Myslajek, MT Neiber, V Nicolas, M Novo, P Ozretic, F Palero, L Parvulescu, M Pascual, OS Paulo, M Pavlek, C Pegueroles, L Pellissier, G Pesole, CR Primmer, A Riesgo, L Ruber, D Rubolini, D Salvi, O Seehausen, M Seidel, B Studer, S Theodoridis, M Thines, L Urban, A Vasemägi, A Vella, N Vella, SC Vernes, C Vernesi, DR Vieites, CW Wheat, G Worheide, Y Wurm and G Zammit. How genomics can help biodiversity conservation. Trends in Genetics 39, 545-559. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.jgg.2023.03.006">L Silva, T Mendes, L Ramos, <b>G Zhang</b> and A Antunes. Parallel evolution of fish bi-modal breathing and expansion of olfactory receptor (OR) genes: toward a universal ORs nomenclature. Journal of Genetics and Genomics (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41559-023-02055-3">RR Germain, S Feng, G Chen, GR Graves, JA Tobias, C Rahbek, F Lei, J Fjeldsa, PA Hosner, MTP Gilbert, <b>G Zhang</b> and D Nogues-Bravo. Species-specific traits mediate avian demographic responses under past climate change. Nature Ecology & Evolution 7, 862-872. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.2201945119">RR Germain, S Feng, L Buffan, CP Carmona, G Chen, GR Graves, JA Tobias, C Rahbek, F Lei, J Fjeldsa, PA Hosner, MTP Gilbert, <b>G Zhang</b> and D Nogues-Bravo. Changes in the functional diversity of modern bird species over the last million years. Proceedings of the National Academy of Sciences 120, e2201945119. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41586-023-05752-y">LA Bergeron, S Besenbacher, J Zheng, P Li, MF Bertelsen, B Quintard, JI Hoffman, Z Li, J St. Leger, C Shao, J Stiller, MTP Gilbert, MH Schierup and <b>G Zhang</b>. Evolution of the germline mutation rate across vertebrates. Nature 615, 285-291. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.celrep.2023.112112">P Zhang, Y Zhu, Q Guo, J Li, X Zhan, H Yu, N Xie, H Tan, N Lundholm, L Garcia-Cuetos, MD Martin, MA Subirats, Y-H Su, I Ruiz-Trillo, MQ Martindale, J-K Yu, MTP Gilbert, <b>G Zhang</b> and Q Li. On the origin and evolution of RNA editing in metazoans. Cell Reports 42, 112112. (2023)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41576-022-00547-9"><b>G Zhang</b>. The mutation rate as an evolving trait. Nature Reviews Genetics 24, 3-3. (2023)</a></p></li>

<h2>-- 2022年研究成果 --</h2>
<li><p><a href="https://doi.org/10.24272/j.issn.2095-8137.2022.001">DD Wu, XG Qi, L Yu, M Li, ZJ Liu, AD Yoder, C Roos, T Hayakawa, J Rogers, T Marques-Bonet, B Su, YG Yao, YP Zhang and <b>G Zhang</b>. Initiation of the Primate Genome Project. Zoological Research 43, 147-149. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.jgg.2021.06.013">Z Wang, J Zhang, X Xu, C Witt, Y Deng, G Chen, G Meng, S Feng, L Xu, T Szekely, <b>G Zhang</b> and Q Zhou. Phylogeny and sex chromosome evolution of Palaeognathae. Journal of Genetics and Genomics 49, 109-119. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1186/s12915-022-01427-8">H Toh, C Yang, G Formenti, K Raja, L Yan, A Tracey, W Chow, K Howe, LA Bergeron, <b>G Zhang</b>, B Haase, J Mountcastle, O Fedrigo, J Fogg, B Kirilenko, C Munegowda, M Hiller, A Jain, D Kihara, A Rhie, AM Phillippy, SA Swanson, P Jiang, DO Clegg, ED Jarvis, JA Thomson, R Stewart, MJP Chaisson and YV Bukhman. A haplotype-resolved genome assembly of the Nile rat facilitates exploration of the genetic basis of diabetes. BMC Biology 20, 245. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41559-022-01884-y">B Qiu, X Dai, P Li, RS Larsen, R Li, AL Price, G Ding, MJ Texada, X Zhang, D Zuo, Q Gao, W Jiang, T Wen, L Pontieri, C Guo, K Rewitz, Q Li, W Liu, JJ Boomsma and <b>G Zhang</b>. Canalized gene expression during development mediates caste differentiation in ants. Nature Ecology & Evolution 6, 1753-1765. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41592-022-01512-4">Y Mao and <b>G Zhang</b>. A complete, telomere-to-telomere human genome sequence presents new opportunities for evolutionary genomics. Nature Methods 19, 635-638. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.cub.2022.02.027">J Lin, D Duchene, C Caroe, O Smith, MM Ciucani, J Niemann, D Richmond, AD Greenwood, R Macphee, <b>G Zhang</b>, S Gopalakrishnan and MTP Gilbert. Probing the genomic limits of de-extinction in the Christmas Island rat. Current Biology 32, 1650-1656.e1653. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.3390/v14071370">Y Li, <b>G Zhang</b> and J Cui. Origin and Deep Evolution of Human Endogenous Retroviruses in Pan-Primates. Viruses 14, 7. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41597-022-01680-0">X Li, R Gao, G Chen, AL Price, DB Oksnebjerg, PA Hosner, Y Zhou, <b>G Zhang</b> and S Feng. Draft genome assemblies of four manakins. Scientific Data 9, 564. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41559-022-01784-1">Q Li, M Wang, P Zhang, Y Liu, Q Guo, Y Zhu, T Wen, X Dai, X Zhang, M Nagel, BH Dethlefsen, N Xie, J Zhao, W Jiang, L Han, L Wu, W Zhong, Z Wang, X Wei, W Dai, L Liu, X Xu, H Lu, H Yang, J Wang, JJ Boomsma, C Liu, <b>G Zhang</b> and W Liu. A single-cell transcriptomic atlas tracking the neural basis of division of labour in an ant superorganism. Nature Ecology & Evolution 6, 1191-1204. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1111/1755-0998.13561">F Li, RV Rane, V Luria, Z Xiong, J Chen, Z Li, RA Catullo, PC Griffin, M Schiffer, S Pearce, SF Lee, K Mcelroy, A Stocker, J Shirriffs, F Cockerell, C Coppin, CM Sgro, A Karger, JW Cain, JA Weber, G Santpere, MW Kirschner, AA Hoffmann, JG Oakeshott and <b>G Zhang</b>. Phylogenomic analyses of the genus Drosophila reveals genomic signals of climate adaptation. Molecular Ecology Resources 22, 1559-1581. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.2115635118">HA Lewin, S Richards, E Lieberman Aiden, ML Allende, JM Archibald, M Balint, KB Barker, B Baumgartner, K Belov, G Bertorelle, ML Blaxter, J Cai, ND Caperello, K Carlson, JC Castilla-Rubio, S-M Chaw, L Chen, AK Childers, JA Coddington, DA Conde, M Corominas, KA Crandall, AJ Crawford, F Dipalma, R Durbin, TE Ebenezer, SV Edwards, O Fedrigo, P Flicek, G Formenti, RA Gibbs, MTP Gilbert, MM Goldstein, JM Graves, HT Greely, IV Grigoriev, KJ Hackett, N Hall, D Haussler, KM Helgen, CJ Hogg, S Isobe, KS Jakobsen, A Janke, ED Jarvis, WE Johnson, SJM Jones, EK Karlsson, PJ Kersey, J-H Kim, WJ Kress, S Kuraku, MKN Lawniczak, JH Leebens-Mack, X Li, K Lindblad-Toh, X Liu, JV Lopez, T Marques-Bonet, S Mazard, JaK Mazet, CJ Mazzoni, EW Myers, RJ Oneill, S Paez, H Park, GE Robinson, C Roquet, OA Ryder, JSM Sabir, HB Shaffer, TM Shank, JS Sherkow, PS Soltis, B Tang, L Tedersoo, M Uliano-Silva, K Wang, X Wei, R Wetzer, JL Wilson, X Xu, H Yang, AD Yoder and <b>G Zhang</b>. The Earth BioGenome Project 2020: Starting the clock. Proceedings of the National Academy of Sciences 119, e2115635118. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.2115639118">MKN Lawniczak, R Durbin, P Flicek, K Lindblad-Toh, X Wei, JM Archibald, WJ Baker, K Belov, ML Blaxter, T Marques Bonet, AK Childers, JA Coddington, KA Crandall, AJ Crawford, RP Davey, F Di Palma, Q Fang, W Haerty, N Hall, KJ Hoff, K Howe, ED Jarvis, WE Johnson, RN Johnson, PJ Kersey, X Liu, JV Lopez, EW Myers, OV Pettersson, AM Phillippy, MF Poelchau, KD Pruitt, A Rhie, JC Castilla-Rubio, SK Sahu, NA Salmon, PS Soltis, D Swarbreck, F Thibaud-Nissen, S Wang, JL Wegrzyn, <b>G Zhang</b>, H Zhang, HA Lewin and S Richards. Standards recommendations for the Earth BioGenome Project. Proceedings of the National Academy of Sciences 119, e2115639118. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1186/s13059-022-02765-0">J Kim, C Lee, BJ Ko, DA Yoo, S Won, AM Phillippy, O Fedrigo, <b>G Zhang</b>, K Howe, J Wood, R Durbin, G Formenti, S Brown, L Cantin, CV Mello, S Cho, A Rhie, H Kim and ED Jarvis. False gene and chromosome losses in genome assemblies caused by GC content variation and repeats. Genome Biology 23, 204. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1126/sciadv.abo0099">Y Ji, S Feng, L Wu, Q Fang, A Bruniche-Olsen, JA Dewoody, Y Cheng, D Zhang, Y Hao, G Song, Y Qu, A Suh, <b>G Zhang</b>, SJ Hackett and F Lei. Orthologous microsatellites, transposable elements, and DNA deletions correlate with generation time and body mass in neoavian birds. Science Advances 8, eabo0099. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41586-022-05325-5">ED Jarvis, G Formenti, A Rhie, A Guarracino, C Yang, J Wood, A Tracey, F Thibaud-Nissen, MR Vollger, D Porubsky, H Cheng, M Asri, GA Logsdon, P Carnevali, MJP Chaisson, C-S Chin, S Cody, J Collins, P Ebert, M Escalona, O Fedrigo, RS Fulton, LL Fulton, S Garg, JL Gerton, J Ghurye, A Granat, RE Green, W Harvey, P Hasenfeld, A Hastie, M Haukness, EB Jaeger, M Jain, M Kirsche, M Kolmogorov, JO Korbel, S Koren, J Korlach, J Lee, D Li, T Lindsay, J Lucas, F Luo, T Marschall, MW Mitchell, J Mcdaniel, F Nie, HE Olsen, ND Olson, T Pesout, T Potapova, D Puiu, A Regier, J Ruan, SL Salzberg, AD Sanders, MC Schatz, A Schmitt, VA Schneider, S Selvaraj, K Shafin, A Shumate, NO Stitziel, C Stober, J Torrance, J Wagner, J Wang, A Wenger, C Xiao, AV Zimin, <b>G Zhang</b>, T Wang, H Li, E Garrison, D Haussler, I Hall, JM Zook, EE Eichler, AM Phillippy, B Paten, K Howe, KH Miga and C Human Pangenome Reference. Semi-automated assembly of high-quality diploid human reference genomes. Nature 611, 519-531. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.tree.2021.11.008">G Formenti, K Theissinger, C Fernandes, I Bista, A Bombarely, C Bleidorn, C Ciofi, A Crottini, JA Godoy, J Hoglund, J Malukiewicz, A Mouton, RA Oomen, S Paez, PJ Palsboll, C Pampoulie, MJ Ruiz-Lopez, H Svardal, C Theofanopoulou, J De Vries, A-M Waldvogel, <b>G Zhang</b>, CJ Mazzoni, ED Jarvis, M Balint, G Formenti, K Theissinger, C Fernandes, I Bista, A Bombarely, C Bleidorn, F Čiampor, C Ciofi, A Crottini, JA Godoy, J Hoglund, J Malukiewicz, A Mouton, RA Oomen, S Paez, P Palsboll, C Pampoulie, MJ Ruiz-Lopez, H Svardal, C Theofanopoulou, J De Vries, A-M Waldvogel, G Zhang, CJ Mazzoni, E Jarvis, M Balint, SA Aghayan, TS Alioto, I Almudi, N Alvarez, PC Alves, IR Amorim, A Antunes, P Arribas, P Baldrian, PR Berg, G Bertorelle, A Bohne, A Bonisoli-Alquati, LL Bostjancic, B Boussau, CM Breton, E Buzan, PF Campos, C Carreras, LF Castro, LJ Chueca, E Conti, R Cook-Deegan, D Croll, MV Cunha, F Delsuc, AB Dennis, D Dimitrov, R Faria, A Favre, OD Fedrigo, R Fernandez, GF Ficetola, J-F Flot, T Gabaldon, DR Galea Agius, GR Gallo, AM Giani, MTP Gilbert, T Grebenc, K Guschanski, R Guyot, B Hausdorf, O Hawlitschek, PD Heintzman, B Heinze, M Hiller, M Husemann, A Iannucci, I Irisarri, KS Jakobsen, S Jentoft, P Klinga, A Kloch, CF Kratochwil, H Kusche, KKS Layton, JA Leonard, E Lerat, G Liti, T Manousaki, T Marques-Bonet, P Matos-Maravi, M Matschiner, F Maumus, AM Mc Cartney, S Meiri, J Melo-Ferreira, X Mengual, MT Monaghan, M Montagna, RW Myslajek, MT Neiber, V Nicolas, M Novo, P Ozretic, F Palero, L Parvulescu, M Pascual, OS Paulo, M Pavlek, C Pegueroles, L Pellissier, G Pesole, CR Primmer, A Riesgo, L Ruber, D Rubolini, D Salvi, O Seehausen, M Seidel, S Secomandi, B Studer, S Theodoridis, M Thines, L Urban, A Vasemägi, A Vella, N Vella, SC Vernes, C Vernesi, DR Vieites, RM Waterhouse, CW Wheat, G Worheide, Y Wurm and G Zammit. The era of reference genomes in conservation genomics. Trends in Ecology & Evolution 37, 197-202. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1093/molbev/msab354">SR Fiddaman, M Vinkler, SG Spiro, H Levy, CA Emerling, AC Boyd, EA Dimopoulos, JA Vianna, TL Cole, H Pan, M Fang, <b>G Zhang</b>, T Hart, LaF Frantz and AL Smith. Adaptation and Cryptic Pseudogenization in Penguin Toll-Like Receptors. Molecular Biology and Evolution 39, msab354. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1101/gr.276395.121">L Ferrandez-Peral, X Zhan, M Alvarez-Estape, C Chiva, P Esteller-Cucala, R Garcia-Perez, E Julia, E Lizano, O Fornas, E Sabido, Q Li, T Marques-Bonet, D Juan and <b>G Zhang</b>. Transcriptome innovations in primates revealed by single-molecule long-read sequencing. Genome research 32, 1448-1462. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.cell.2022.03.034">S Feng, M Bai, I Rivas-Gonzalez, C Li, S Liu, Y Tong, H Yang, G Chen, D Xie, KE Sears, LM Franco, JD Gaitan-Espitia, RF Nespolo, WE Johnson, H Yang, PA Brandies, CJ Hogg, K Belov, MB Renfree, KM Helgen, JJ Boomsma, MH Schierup and <b>G Zhang</b>. Incomplete lineage sorting and phenotypic evolution in marsupials. Cell 185, 1646-1660.e1618. (2022)</a></p></li>
<li><p><a href="https://www.frontiersin.org/articles/10.3389/fphys.2022.852357">G Ding, Q Gao, J Chen, J Zhao, <b>G Zhang</b> and W Liu. Validation of Potential Reference Genes for Real-Time qPCR Analysis in Pharaoh Ant, Monomorium pharaonis (Hymenoptera: Formicidae). Frontiers in Physiology 13, (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.2109326119">B Demarchi, J Stiller, A Grealy, M Mackie, Y Deng, T Gilbert, J Clarke, LJ Legendre, R Boano, T Sicheritz-Ponten, J Magee, <b>G Zhang</b>, M Bunce, MJ Collins and G Miller. Ancient proteins resolve controversy over the identity of Genyornis eggshell. Proceedings of the National Academy of Sciences 119, e2109326119. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41467-022-31508-9">TL Cole, C Zhou, M Fang, H Pan, DT Ksepka, SR Fiddaman, CA Emerling, DB Thomas, X Bi, Q Fang, MR Ellegaard, S Feng, AL Smith, TA Heath, AJD Tennyson, PG Borboroglu, JR Wood, PW Hadden, S Grosser, C-A Bost, Y Cherel, T Mattern, T Hart, M-HS Sinding, LD Shepherd, RA Phillips, P Quillfeldt, JF Masello, JL Bouzat, PG Ryan, DR Thompson, U Ellenberg, P Dann, G Miller, P Dee Boersma, R Zhao, MTP Gilbert, H Yang, D-X Zhang and <b>G Zhang</b>. Genomic insights into the secondary aquatic transition of penguins. Nature Communications 13, 3912. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1111/ibi.13023">EL Cavill, S Gopalakrishnan, LC Puetz, AM Ribeiro, SST Mak, RR Da Fonseca, G Pacheco, B Dunlop, W Accouche, N Shah, A Zora, L Calabrese, M Genner, G Jones, C Guo, <b>G Zhang</b> and MTP Gilbert. Conservation genomics of the endangered Seychelles Magpie-Robin (Copsychus sechellarum): a unique insight into the history of a precious endemic bird. IBIS 164, 396-410. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.2115636118">M Blaxter, JM Archibald, AK Childers, JA Coddington, KA Crandall, F Di Palma, R Durbin, SV Edwards, JaM Graves, KJ Hackett, N Hall, ED Jarvis, RN Johnson, EK Karlsson, WJ Kress, S Kuraku, MKN Lawniczak, K Lindblad-Toh, JV Lopez, NA Moran, GE Robinson, OA Ryder, B Shapiro, PS Soltis, T Warnow, <b>G Zhang</b> and HA Lewin. Why sequence all eukaryotes? Proceedings of the National Academy of Sciences 119, e2115636118. (2022)</a></p></li>
<li><p><a href="https://doi.org/10.7554/eLife.73577">LA Bergeron, S Besenbacher, T Turner, CJ Versoza, RJ Wang, AL Price, E Armstrong, M Riera, J Carlson, H-Y Chen, MW Hahn, K Harris, AS Kleppe, EH Lopez-Nandam, P Moorjani, SP Pfeifer, GP Tiley, AD Yoder, <b>G Zhang</b> and MH Schierup. The Mutationathon highlights the importance of reaching standardization in estimates of pedigree-based germline mutation rates. eLife 11, e73577. (2022)</a></p></li>

<h2>-- 2021年研究成果 --</h2>
<li><p><a href="https://doi.org/10.1038/s41586-020-03039-0">Y Zhou, L Shearwin-Whyatt, J Li, Z Song, T Hayakawa, D Stevens, JC Fenelon, E Peel, Y Cheng, F Pajpach, N Bradley, H Suzuki, M Nikaido, J Damas, T Daish, T Perry, Z Zhu, Y Geng, A Rhie, Y Sims, J Wood, B Haase, J Mountcastle, O Fedrigo, Q Li, H Yang, J Wang, SD Johnston, AM Phillippy, K Howe, ED Jarvis, OA Ryder, H Kaessmann, P Donnelly, J Korlach, HA Lewin, J Graves, K Belov, MB Renfree, F Grutzner, Q Zhou and <b>G Zhang</b>. Platypus and echidna genomes reveal mammalian biology and evolution. Nature 592, 756-762. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41586-021-03535-x">C Yang, Y Zhou, S Marcus, G Formenti, LA Bergeron, Z Song, X Bi, J Bergman, MMC Rousselle, C Zhou, L Zhou, Y Deng, M Fang, D Xie, Y Zhu, S Tan, J Mountcastle, B Haase, J Balacco, J Wood, W Chow, A Rhie, M Pippel, MM Fabiszak, S Koren, O Fedrigo, WA Freiwald, K Howe, H Yang, AM Phillippy, MH Schierup, ED Jarvis and <b>G Zhang</b>. Evolutionary and biomedical insights from a marmoset diploid genome assembly. Nature 594, 227-233. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.tig.2021.05.009">C Yang and <b>G Zhang</b>. Callithrix jacchus (the common marmoset). Trends in Genetics 37, 948-949. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.24272/j.issn.2095-8137.2020.175">Z Wang, G Chen, <b>G Zhang</b> and Q Zhou. Dynamic evolution of transposable elements, demographic history, and gene content of paleognathous birds. Zoological Research 42, 51. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1186/s12915-021-01052-x">M-S Wang, J-J Zhang, X Guo, M Li, R Meyer, H Ashari, Z-Q Zheng, S Wang, M-S Peng, Y Jiang, M Thakur, C Suwannapoom, A Esmailizadeh, NY Hirimuthugoda, MSA Zein, S Kusza, H Kharrati-Koopaee, L Zeng, Y-M Wang, T-T Yin, M-M Yang, M-L Li, X-M Lu, E Lasagna, S Ceccobelli, HGTN Gunwardana, TM Senasig, S-H Feng, H Zhang, AKFH Bhuiyan, MS Khan, GLLP Silva, LT Thuy, OA Mwai, MNM Ibrahim, <b>G Zhang</b>, K-X Qu, O Hanotte, B Shapiro, M Bosse, D-D Wu, J-L Han and Y-P Zhang. Large-scale genomic analysis reveals the genetic cost of chicken domestication. BMC Biology 19, 118. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.cell.2021.01.047">K Wang, J Wang, C Zhu, L Yang, Y Ren, J Ruan, G Fan, J Hu, W Xu, X Bi, Y Zhu, Y Song, H Chen, T Ma, R Zhao, H Jiang, B Zhang, C Feng, Y Yuan, X Gan, Y Li, H Zeng, Q Liu, Y Zhang, F Shao, S Hao, H Zhang, X Xu, X Liu, D Wang, M Zhu, <b>G Zhang</b>, W Zhao, Q Qiu, S He and W Wang. African lungfish genome sheds light on the vertebrate water-to-land transition. Cell 184, 1362-1376.e1318. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.2023836118">K Thorup, L Pedersen, RR Da Fonseca, B Naimi, D Nogues-Bravo, M Krapp, A Manica, M Willemoes, S Sjoberg, S Feng, G Chen, A Rey-Iglesia, PF Campos, R Beyer, MB Araujo, AJ Hansen, <b>G Zhang</b>, AP Tottrup and C Rahbek. Response of an Afro-Palearctic bird migrant to glaciation cycles. Proceedings of the National Academy of Sciences 118, e2023836118. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41467-021-23950-y">L Schrader, H Pan, M Bollazzi, M Schiott, FJ Larabee, X Bi, Y Deng, <b>G Zhang</b>, JJ Boomsma and C Rabeling. Publisher Correction: Relaxed selection underlies genome erosion in socially parasitic ant species. Nature Communications 12, 3821. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1111/mec.16043">F Sanchez-Barreiro, S Gopalakrishnan, J Ramos-Madrigal, MV Westbury, M De Manuel, A Margaryan, MM Ciucani, FG Vieira, Y Patramanis, DC Kalthoff, Z Timmons, T Sicheritz-Pontn, L Dalen, OA Ryder, <b>G Zhang</b>, T Marques-Bonet, Y Moodley and MTP Gilbert. Historical population declines prompted significant genomic erosion in the northern and southern white rhinoceros (Ceratotherium simum). Molecular Ecology 30, 6355-6369. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41586-021-03451-0">A Rhie, SA Mccarthy, O Fedrigo, J Damas, G Formenti, S Koren, M Uliano-Silva, W Chow, A Fungtammasan, J Kim, C Lee, BJ Ko, M Chaisson, GL Gedman, LJ Cantin, F Thibaud-Nissen, L Haggerty, I Bista, M Smith, B Haase, J Mountcastle, S Winkler, S Paez, J Howard, SC Vernes, TM Lama, F Grutzner, WC Warren, CN Balakrishnan, D Burt, JM George, MT Biegler, D Iorns, A Digby, D Eason, B Robertson, T Edwards, M Wilkinson, G Turner, A Meyer, AF Kautt, P Franchini, HW Detrich, H Svardal, M Wagner, GJP Naylor, M Pippel, M Malinsky, M Mooney, M Simbirsky, BT Hannigan, T Pesout, M Houck, A Misuraca, SB Kingan, R Hall, Z Kronenberg, I Sovic, C Dunn, Z Ning, A Hastie, J Lee, S Selvaraj, RE Green, NH Putnam, I Gut, J Ghurye, E Garrison, Y Sims, J Collins, S Pelan, J Torrance, A Tracey, J Wood, RE Dagnew, D Guan, SE London, DF Clayton, CV Mello, SR Friedrich, PV Lovell, E Osipova, FO Al-Ajli, S Secomandi, H Kim, C Theofanopoulou, M Hiller, Y Zhou, RS Harris, KD Makova, P Medvedev, J Hoffman, P Masterson, K Clark, F Martin, K Howe, P Flicek, BP Walenz, W Kwak, H Clawson, M Diekhans, L Nassar, B Paten, RHS Kraus, AJ Crawford, MTP Gilbert, <b>G Zhang</b>, B Venkatesh, RW Murphy, K-P Koepfli, B Shapiro, WE Johnson, F Di Palma, T Marques-Bonet, EC Teeling, T Warnow, JM Graves, OA Ryder, D Haussler, SJ Obrien, J Korlach, HA Lewin, K Howe, EW Myers, R Durbin, AM Phillippy and ED Jarvis. Towards complete and error-free genome assemblies of all vertebrate species. Nature 592, 737-746. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.2012398118">Y Qu, C Chen, X Chen, Y Hao, H She, M Wang, PGP Ericson, H Lin, T Cai, G Song, C Jia, C Chen, H Zhang, J Li, L Liang, T Wu, J Zhao, Q Gao, <b>G Zhang</b>, W Zhai, C Zhang, YE Zhang and F Lei. The evolution of ancestral and species-specific adaptations in snowfinches at the Qinghai Tibet Plateau. Proceedings of the National Academy of Sciences 118, e2012398118. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1002/ggn2.202100018">LC Puetz, TO Delmont, O Aizpurua, C Guo, <b>G Zhang</b>, R Katajamaa, P Jensen and MTP Gilbert. Gut Microbiota Linked with Reduced Fear of Humans in Red Junglefowl Has Implications for Early Domestication. Advanced Genetics 2, 2100018. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1007/s11427-020-1828-8">X Pan, Y Cai, Z Li, X Chen, R Heller, N Wang, Y Wang, C Zhao, Y Wang, H Xu, S Li, M Li, C Li, S Hu, H Li, K Wang, L Chen, B Wei, Z Zheng, W Fu, Y Yang, T Zhang, Z Hou, Y Yan, X Lv, W Sun, X Li, S Huang, L Liu, S Mao, W Liu, J Hua, Z Li, <b>G Zhang</b>, Y Chen, X Wang, Q Qiu, BP Dalrymple, W Wang and Y Jiang. Modes of genetic adaptations underlying functional innovations in the rumen. Science China Life Sciences 64, 1-21. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1111/mec.16006">PA Obrien, N Andreakis, S Tan, DJ Miller, NS Webster, <b>G Zhang</b> and DG Bourne. Testing cophylogeny between coral reef invertebrates and their bacterial and archaeal symbionts. Molecular Ecology 30, 3768-3782. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1002/ajp.23255">AD Melin, JD Orkin, MC Janiak, A Valenzuela, L Kuderna, F Marrone Iii, H Ramangason, JE Horvath, C Roos, AC Kitchener, CC Khor, WK Lim, JGH Lee, P Tan, G Umapathy, M Raveendran, R Alan Harris, I Gut, M Gut, E Lizano, T Nadler, D Zinner, MD Le, S Manu, CJ Rabarivola, A Zaramody, N Andriaholinirina, SE Johnson, ED Jarvis, O Fedrigo, D Wu, <b>G Zhang</b>, KK-H Farh, J Rogers, T Marques-Bonet, A Navarro, D Juan, PS Arora and JP Higham. Variation in predicted COVID-19 risk among lemurs and lorises. American Journal of Primatology 83, e23255. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.cell.2021.07.032">S Liu, MV Westbury, N Dussex, KJ Mitchell, M-HS Sinding, PD Heintzman, DA Duchene, JD Kapp, J Von Seth, H Heiniger, F Sanchez-Barreiro, A Margaryan, R Andre-Olsen, B De Cahsan, G Meng, C Yang, L Chen, T Van Der Valk, Y Moodley, K Rookmaaker, MW Bruford, O Ryder, C Steiner, LGR Bruins-Van Sonsbeek, S Vartanyan, C Guo, A Cooper, P Kosintsev, I Kirillova, AM Lister, T Marques-Bonet, S Gopalakrishnan, RR Dunn, ED Lorenzen, B Shapiro, <b>G Zhang</b>, P-O Antoine, L Dalen and MTP Gilbert. Ancient and modern genomes unravel the evolutionary history of the rhinoceros family. Cell 184, 4874-4885.e4816. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1101/gr.271569.120">J Liu, Z Wang, J Li, L Xu, J Liu, S Feng, C Guo, S Chen, Z Ren, J Rao, K Wei, Y Chen, ED Jarvis, <b>G Zhang</b> and Q Zhou. A new emu genome illuminates the evolution of genome configuration and nuclear architecture of avian chromosomes. Genome research 31, 497-511. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1093/gigascience/giaa142">J Li, J Zhang, J Liu, Y Zhou, C Cai, L Xu, X Dai, S Feng, C Guo, J Rao, K Wei, ED Jarvis, Y Jiang, Z Zhou, <b>G Zhang</b> and Q Zhou. A new duck genome reveals conserved and convergently evolved chromosome architectures of birds and mammals. GigaScience 10, giaa142. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1371/journal.pone.0260119">PF Lai, K Lei, X Zhan, G Sooranna, JKH Li, EX Georgiou, A Das, N Singh, Q Li, Z Stanfield, <b>G Zhang</b>, RM Tribe, S Mesiano and MR Johnson. Labour classified by cervical dilatation & fetal membrane rupture demonstrates differential impact on RNA-seq data for human myometrium tissues. PLOS ONE 16, e0260119. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.2100338118">TR Feuerborn, A Carmagnini, RJ Losey, T Nomokonova, A Askeyev, I Askeyev, O Askeyev, EE Antipina, M Appelt, OP Bachura, F Beglane, DG Bradley, KG Daly, S Gopalakrishnan, K Murphy Gregersen, C Guo, AV Gusev, C Jones, PA Kosintsev, YV Kuzmin, V Mattiangeli, AR Perri, AV Plekhanov, J Ramos-Madrigal, AL Schmidt, D Shaymuratova, O Smith, LV Yavorskaya, <b>G Zhang</b>, E Willerslev, M Meldgaard, MTP Gilbert, G Larson, L Dalen, AJ Hansen, M-HS Sinding and L Frantz. Modern Siberian dog ancestry was shaped by several thousand years of Eurasian-wide trade and human dispersal. Proceedings of the National Academy of Sciences 118, e2100338118. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41586-021-03473-8">S Feng, J Stiller, Y Deng, J Armstrong, Q Fang, AH Reeve, D Xie, G Chen, C Guo, BC Faircloth, B Petersen, Z Wang, Q Zhou, M Diekhans, W Chen, S Andreu-Sanchez, A Margaryan, JT Howard, C Parent, G Pacheco, M-HS Sinding, L Puetz, E Cavill, AM Ribeiro, L Eckhart, J Fjeldsa, PA Hosner, RT Brumfield, L Christidis, MF Bertelsen, T Sicheritz-Ponten, DT Tietze, BC Robertson, G Song, G Borgia, S Claramunt, IJ Lovette, SJ Cowen, P Njoroge, JP Dumbacher, OA Ryder, J Fuchs, M Bunce, DW Burt, J Cracraft, G Meng, SJ Hackett, PG Ryan, KA Jonsson, IG Jamieson, RR Da Fonseca, EL Braun, P Houde, S Mirarab, A Suh, B Hansson, S Ponnikas, H Sigeman, M Stervander, PB Frandsen, H Van Der Zwan, R Van Der Sluis, C Visser, CN Balakrishnan, AG Clark, JW Fitzpatrick, R Bowman, N Chen, A Cloutier, TB Sackton, SV Edwards, DJ Foote, SB Shakya, FH Sheldon, A Vignal, AER Soares, B Shapiro, J Gonzalez-Solis, J Ferrer-Obiol, J Rozas, M Riutort, A Tigano, V Friesen, L Dalen, AO Urrutia, T Szekely, Y Liu, MG Campana, A Corvelo, RC Fleischer, KM Rutherford, NJ Gemmell, N Dussex, H Mouritsen, N Thiele, K Delmore, M Liedvogel, A Franke, MP Hoeppner, O Krone, AM Fudickar, B Mila, ED Ketterson, AE Fidler, G Friis, AM Parody-Merino, PF Battley, MP Cox, NCB Lima, F Prosdocimi, TL Parchman, BA Schlinger, BA Loiselle, JG Blake, HC Lim, LB Day, MJ Fuxjager, MW Baldwin, MJ Braun, M Wirthlin, RB Dikow, TB Ryder, G Camenisch, LF Keller, JM Dacosta, ME Hauber, MIM Louder, CC Witt, JA Mcguire, J Mudge, LC Megna, MD Carling, B Wang, SA Taylor, G Del-Rio, A Aleixo, ATR Vasconcelos, CV Mello, JT Weir, D Haussler, Q Li, H Yang, J Wang, F Lei, C Rahbek, MTP Gilbert, GR Graves, ED Jarvis, B Paten and <b>G Zhang</b>. Author Correction: Dense sampling of bird diversity increases power of comparative genomics. Nature 592, E24-E24. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.cub.2021.09.059">MM Ciucani, JK Jensen, M-HS Sinding, O Smith, SB Lucenti, E Rosengren, L Rook, C Tuveri, M Arca, E Cappellini, M Galaverni, E Randi, C Guo, <b>G Zhang</b>, T Sicheritz-Ponten, L Dalen, MTP Gilbert and S Gopalakrishnan. Evolutionary history of the extinct Sardinian dhole. Current Biology 31, 5571-5579.e5576. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.24272/j.issn.2095-8137.2021.066">X Bi and <b>G Zhang</b>. Ancestral developmental potentials in early bony fish contributed to vertebrate water-to-land transition. Zoological Research 42, 135. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.cell.2021.01.046">X Bi, K Wang, L Yang, H Pan, H Jiang, Q Wei, M Fang, H Yu, C Zhu, Y Cai, Y He, X Gan, H Zeng, D Yu, Y Zhu, H Jiang, Q Qiu, H Yang, YE Zhang, W Wang, M Zhu, S He and <b>G Zhang</b>. Tracing the genetic footprints of vertebrate landing in non-teleost ray-finned fishes. Cell 184, 1377-1391.e1314. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1093/gigascience/giab072">LA Bergeron, S Besenbacher, MH Schierup and <b>G Zhang</b>. Studying mutation rate evolution in primates a need for systematic comparison of computational pipelines. GigaScience 10, giab072. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1093/gigascience/giab029">LA Bergeron, S Besenbacher, J Bakker, J Zheng, P Li, G Pacheco, M-HS Sinding, M Kamilari, MTP Gilbert, MH Schierup and <b>G Zhang</b>. The germline mutational process in rhesus macaque and its implications for phylogenetic dating. GigaScience 10, giab029. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.1093/gigascience/giaa161">S Andreu-Sanchez, W Chen, J Stiller and <b>G Zhang</b>. Multiple origins of a frameshift insertion in a mitochondrial gene in birds and turtles. GigaScience 10, giaa161. (2021)</a></p></li>
<li><p><a href="https://doi.org/10.25849/myrmecol.news_031:201">RMM Adams, RS Larsen, N Stylianidi, D Cheung, B Qiu, SK Murray, <b>G Zhang</b> and JJ Boomsma. Hairs distinguish castes and sexes: identifying the early ontogenetic building blocks of a fungus-farming superorganism (Hymenoptera: Formicidae). Myrmecological News 31, 201-216. (2021)</a></p></li>  

<h2>-- 2020年研究成果 --</h2>
<li><p><a href="https://doi.org/10.1126/sciadv.aaz5216">Z Zheng, X Wang, M Li, Y Li, Z Yang, X Wang, X Pan, M Gong, Y Zhang, Y Guo, Y Wang, J Liu, Y Cai, Q Chen, M Okpeku, L Colli, D Cai, K Wang, S Huang, TS Sonstegard, A Esmailizadeh, W Zhang, T Zhang, Y Xu, N Xu, Y Yang, J Han, L Chen, J Lesur, KG Daly, DG Bradley, R Heller, <b>G Zhang</b>, W Wang, Y Chen and Y Jiang. The origin of domestication genes in goats. Science Advances 6, eaaz5216. (2020)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.isci.2020.101640">P Zhang, Y Zhao, C Li, M Lin, L Dong, R Zhang, M Liu, K Li, H Zhang, X Liu, Y Zhang, Y Yuan, H Liu, I Seim, S Sun, X Du, Y Chang, F Li, S Liu, SM-Y Lee, K Wang, D Wang, X Wang, MR Mcgowen, TA Jefferson, MT Olsen, J Stiller, <b>G Zhang</b>, X Xu, H Yang, G Fan, X Liu and S Li. An Indo-Pacific Humpback Dolphin Genome Reveals Insights into Chromosome Evolution and the Demography of a Vulnerable Species. iScience 23, 101640. (2020)</a></p></li>
<li><p><a href="https://www.zoores.ac.cn/en/article/id/3992">P Zhang, J Chen, Q Li, L Sheng, Y Gao, B Lu, W Zhu, X Zhan, Y Li, Z Yuan, G Xu, B Qiu, M Yan, C Guo, Y Wang, Y Huang, J Zhang, F Liu, Z Tang, S Lin, DN Cooper, H Yang, J Wang, Y Gao, W Yin, <b>G Zhang</b> and G Yan. Neuroprotectants attenuate hypobaric hypoxia-induced brain injuries in cynomolgus monkeys. Zoological Research 41, 3. (2020)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41597-020-0468-9">C Yang, F Li, Z Xiong, K-P Koepfli, O Ryder, P Perelman, Q Li and <b>G Zhang</b>. A draft genome assembly of spotted hyena, Crocuta crocuta. Scientific Data 7, 126. (2020)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41422-020-0380-z">M-S Wang, M Thakur, M-S Peng, Y Jiang, LaF Frantz, M Li, J-J Zhang, S Wang, J Peters, NO Otecko, C Suwannapoom, X Guo, Z-Q Zheng, A Esmailizadeh, NY Hirimuthugoda, H Ashari, S Suladari, MSA Zein, S Kusza, S Sohrabi, H Kharrati-Koopaee, Q-K Shen, L Zeng, M-M Yang, Y-J Wu, X-Y Yang, X-M Lu, X-Z Jia, Q-H Nie, SJ Lamont, E Lasagna, S Ceccobelli, HGTN Gunwardana, TM Senasige, S-H Feng, J-F Si, H Zhang, J-Q Jin, M-L Li, Y-H Liu, H-M Chen, C Ma, S-S Dai, AKFH Bhuiyan, MS Khan, GLLP Silva, T-T Le, OA Mwai, MNM Ibrahim, M Supple, B Shapiro, O Hanotte, <b>G Zhang</b>, G Larson, J-L Han, D-D Wu and Y-P Zhang. Author Correction: 863 genomes reveal the origin and domestication of chicken. Cell Research 30, 824-825. (2020)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41597-020-0556-x">M Wang, Y Liu, T Wen, W Liu, Q Gao, J Zhao, Z Xiong, Z Wang, W Jiang, Y Yu, L Wu, Y Yuan, X Wei, J Xu, M Cheng, P Zhang, P Li, Y Hou, H Yang, <b>G Zhang</b>, Q Li, C Liu and L Liu. Chromatin accessibility and transcriptome landscapes of Monomorium pharaonis brain. Scientific Data 7, 217. (2020)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41467-020-16454-8">Y Shao, X Wang, J Zhang, M Li, S Wu, X Ma, X Wang, H Zhao, Y Li, H Zhu, D Irwin, D Wang, <b>G Zhang</b>, J Ruan and D Wu. Genome and single-cell RNA-sequencing of the earthworm Eisenia andrei identifies cellular mechanisms underlying regeneration. Nature Communications 11, 2656. (2020)</a></p></li>
<li><p><a href="https://doi.org/10.1093/gigascience/giaa087">M Sandoval-Velasco, JA Rodriguez, C Perez estrada, <b>G Zhang</b>, E Lieberman aiden, MA Marti-Renom, MTP Gilbert and O Smith. Hi-C chromosome conformation capture sequencing of avian genomes using the BGISEQ-500 platform. GigaScience 9, giaa087. (2020)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41396-020-0671-x">PA Obrien, S Tan, C Yang, PR Frade, N Andreakis, HA Smith, DJ Miller, NS Webster, <b>G Zhang</b> and DG Bourne. Diverse coral reef invertebrates exhibit patterns of phylosymbiosis. The ISME Journal 14, 2211-2222. (2020)</a></p></li>
<li><p><a href="https://doi.org/10.1126/sciadv.aaz5772">M Nagel, B Qiu, LE Brandenborg, RS Larsen, D Ning, JJ Boomsma and <b>G Zhang</b>. The gene expression network regulating queen brain remodeling after insemination and its parallel use in ants with reproductive workers. Science Advances 6, eaaz5772. (2020)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.cub.2020.07.046">E Lord, N Dussex, M Kierczak, D Diez-Del-Molino, OA Ryder, DWG Stanton, MTP Gilbert, F Sanchez-Barreiro, <b>G Zhang</b>, M-HS Sinding, ED Lorenzen, E Willerslev, A Protopopov, F Shidlovskiy, S Fedorov, H Bocherens, SKSS Nathan, B Goossens, J Van Der Plicht, YL Chan, S Prost, O Potapova, I Kirillova, AM Lister, PD Heintzman, JD Kapp, B Shapiro, S Vartanyan, A Gotherstrom and L Dalen. Pre-extinction Demographic Stability and Genomic Signatures of Adaptation in the Woolly Rhinoceros. Current Biology 30, 3871-3879.e3877. (2020)</a></p></li>
<li><p><a href="https://www.protocols.io/view/repetitive-element-annotation-for-the-eastern-banj-bgkbjusn">Q Li, Q Guo, Y Zhou, H Tan, T Bertozzi, Y Zhu, J Li, S Donnellan and <b>G Zhang</b>. Repetitive element annotation protocol for the eastern banjo frog. protocols. io (2020)</a></p></li>
<li><p><a href="https://www.protocols.io/view/construction-and-sequencing-of-dna-libraries-on-hi-x54v9m124g3e/v1">Q Li, Q Guo, Y Zhou, H Tan, T Bertozzi, Y Zhu, J Li, S Donnellan and <b>G Zhang</b>. Construction and sequencing of DNA libraries on Hiseq 2000 platform for the eastern banjo frog. protocols. io (2020)</a></p></li>
<li><p><a href="https://www.protocols.io/view/repetitive-element-annotation-for-the-eastern-banj-bgkbjusn">Q Li, Q Guo, Y Zhou, H Tan, T Bertozzi, Y Zhu, J Li, S Donnellan and <b>G Zhang</b>. Repetitive element annotation for the eastern banjo frog genome assembly V. 3. protocols. io (2020)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.2000344117">KM Kapheim, BM Jones, H Pan, C Li, BA Harpur, CF Kent, A Zayed, P Ioannidis, RM Waterhouse, C Kingwell, E Stolle, A Avalos, <b>G Zhang</b>, WO Mcmillan and WT Wcislo. Developmental plasticity shapes social traits and selection in a facultatively eusocial bee. Proceedings of the National Academy of Sciences 117, 13615-13625. (2020)</a></p></li>
<li><p><a href="https://doi.org/10.1111/jeb.13716">J Howe, M Schiott, Q Li, Z Wang, <b>G Zhang</b> and JJ Boomsma. A novel method for using RNA-seq data to identify imprinted genes in social Hymenoptera with multiply mated queens. Journal of Evolutionary Biology 33, 1770-1782. (2020)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.cub.2019.10.066">P Gelabert, M Sandoval-Velasco, A Serres, M De Manuel, P Renom, A Margaryan, J Stiller, T De-Dios, Q Fang, S Feng, S Mañosa, G Pacheco, M Ferrando-Bernal, G Shi, F Hao, X Chen, B Petersen, R-A Olsen, A Navarro, Y Deng, L Dalen, T Marques-Bonet, <b>G Zhang</b>, A Antunes, MTP Gilbert and C Lalueza-Fox. Evolutionary History, Genomic Adaptation to Toxic Diet, and Extinction of the Carolina Parakeet. Current Biology 30, 108-114.e105. (2020)</a></p></li>
<li><p><a href="https://doi.org/10.1093/gigascience/giaa143">Q Gao, Z Xiong, RS Larsen, L Zhou, J Zhao, G Ding, R Zhao, C Liu, H Ran and <b>G Zhang</b>. High-quality chromosome-level genome assembly and full-length transcriptome analysis of the pharaoh ant Monomorium pharaonis. GigaScience 9, giaa143. (2020)</a></p></li>
<li><p><a href="https://doi.org/10.1098/rstb.2019.0584">AK Fotakis, SD Denham, M Mackie, MI Orbegozo, D Mylopotamitaki, S Gopalakrishnan, T Sicheritz-Ponten, JV Olsen, E Cappellini, <b>G Zhang</b>, A Christophersen, MTP Gilbert and AJ Vagene. Multi-omic detection of Mycobacterium leprae in archaeological human dental calculus. Philosophical Transactions of the Royal Society B: Biological Sciences 375, 20190584. (2020)</a></p></li>
<li><p><a href="https://doi.org/10.1111/arcm.12553">AMH Eriksen, H Matthiesen, I Kontopoulos, D Gregory, C Snoeck, <b>G Zhang</b>, MJ Collins and MTP Gilbert. Rapid loss of endogenous DNA in pig bone buried in five different environments. Archaeometry 62, 827-846. (2020)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.1919423117">M De Manuel, R Barnett, M Sandoval-Velasco, N Yamaguchi, F Garrett Vieira, ML Zepeda Mendoza, S Liu, MD Martin, M-HS Sinding, SST Mak, C Caroe, S Liu, C Guo, J Zheng, G Zazula, G Baryshnikov, E Eizirik, K-P Koepfli, WE Johnson, A Antunes, T Sicheritz-Ponten, S Gopalakrishnan, G Larson, H Yang, SJ Obrien, AJ Hansen, <b>G Zhang</b>, T Marques-Bonet and MTP Gilbert. The evolutionary history of extinct and living lions. Proceedings of the National Academy of Sciences 117, 10927-10934. (2020)</a></p></li>
<li><p><a href="https://doi.org/10.1093/gigascience/giz152">RR Da fonseca, A Couto, AM Machado, B Brejova, CB Albertin, F Silva, P Gardner, T Baril, A Hayward, A Campos, AM Ribeiro, I Barrio-Hernandez, H-J Hoving, R Tafur-Jimenez, C Chu, B Frazao, B Petersen, F Penaloza, F Musacchia, GC Alexander, Jr., H Osorio, I Winkelmann, O Simakov, S Rasmussen, MZ Rahman, D Pisani, J Vinther, E Jarvis, <b>G Zhang</b>, JM Strugnell, LFC Castro, O Fedrigo, M Patricio, Q Li, S Rocha, A Antunes, Y Wu, B Ma, R Sanges, T Vinar, B Blagoev, T Sicheritz-Ponten, R Nielsen and MTP Gilbert. A draft genome sequence of the elusive giant squid, Architeuthis dux. GigaScience 9, giz152. (2020)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.cub.2020.09.051">R Barnett, MV Westbury, M Sandoval-Velasco, FG Vieira, S Jeon, G Zazula, MD Martin, SYW Ho, N Mather, S Gopalakrishnan, J Ramos-Madrigal, M De Manuel, ML Zepeda-Mendoza, A Antunes, AC Baez, B De Cahsan, G Larson, SJ Obrien, E Eizirik, WE Johnson, K-P Koepfli, A Wilting, J Fickel, L Dalen, ED Lorenzen, T Marques-Bonet, AJ Hansen, <b>G Zhang</b>, J Bhak, N Yamaguchi and MTP Gilbert. Genomic Adaptations and Evolutionary History of the Extinct Scimitar-Toothed Cat, Homotherium latidens. Current Biology 30, 5018-5025.e5015. (2020)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.1922927117">A Avalos, M Fang, H Pan, A Ramirez Lluch, AE Lipka, SD Zhao, T Giray, GE Robinson, <b>G Zhang</b> and ME Hudson. Genomic regions influencing aggressive behavior in honey bees are defined by colony allele frequencies. Proceedings of the National Academy of Sciences 117, 17135-17141. (2020)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41586-020-2871-y">J Armstrong, G Hickey, M Diekhans, IT Fiddes, AM Novak, A Deran, Q Fang, D Xie, S Feng, J Stiller, D Genereux, J Johnson, VD Marinescu, J Alfoldi, RS Harris, K Lindblad-Toh, D Haussler, E Karlsson, ED Jarvis, <b>G Zhang</b> and B Paten. Progressive Cactus is a multiple-genome aligner for the thousand-genome era. Nature 587, 246-251. (2020)</a></p></li>

<h2>-- 2019年研究成果 --</h2>
<li><p><a href="https://doi.org/10.1038/s41559-019-0850-1">L Xu, G Auer, V Peona, A Suh, Y Deng, S Feng, <b>G Zhang</b>, MPK Blom, L Christidis, S Prost, M Irestedt and Q Zhou. Dynamic evolutionary history and gene content of sex chromosomes across diverse songbirds. Nature Ecology & Evolution 3, 834-844. (2019)</a></p></li>
<li><p><a href="https://doi.org/10.1126/science.aav6335">Y Wang, C Zhang, N Wang, Z Li, R Heller, R Liu, Y Zhao, J Han, X Pan, Z Zheng, X Dai, C Chen, M Dou, S Peng, X Chen, J Liu, M Li, K Wang, C Liu, Z Lin, L Chen, F Hao, W Zhu, C Song, C Zhao, C Zheng, J Wang, S Hu, C Li, H Yang, L Jiang, G Li, M Liu, TS Sonstegard, <b>G Zhang</b>, Y Jiang, W Wang and Q Qiu. Genetic basis of ruminant headgear and rapid antler regeneration. Science 364, eaav6335. (2019)</a></p></li>
<li><p><a href="https://doi.org/10.3390/d11070115">J Stiller and <b>G Zhang</b>. Comparative Phylogenomics, a Stepping Stone for Bird Biodiversity Studies. Diversity 7, 115. (2019)</a></p></li>
<li><p><a href="https://doi.org/10.1111/mec.15176">AM Ribeiro, L Puetz, NB Pattinson, L Dalen, Y Deng, <b>G Zhang</b>, RR Da Fonseca, B Smit and MTP Gilbert. 31 South: The physiology of adaptation to arid conditions in a passerine bird. Molecular Ecology 28, 3709-3721. (2019)</a></p></li>
<li><p><a href="https://doi.org/10.1093/jhered/esy058">RV Rane, DF Clarke, SL Pearce, <b>G Zhang</b>, AA Hoffmann and JG Oakeshott. Detoxification Genes Differ Between Cactus-, Fruit-, and Flower-Feeding Drosophila. Journal of Heredity 110, 80-91. (2019)</a></p></li>
<li><p><a href="https://doi.org/10.1093/gigascience/giz117">H Pan, TL Cole, X Bi, M Fang, C Zhou, Z Yang, DT Ksepka, T Hart, JL Bouzat, LS Argilla, MF Bertelsen, PD Boersma, C-A Bost, Y Cherel, P Dann, SR Fiddaman, P Howard, K Labuschagne, T Mattern, G Miller, P Parker, RA Phillips, P Quillfeldt, PG Ryan, H Taylor, DR Thompson, MJ Young, MR Ellegaard, MTP Gilbert, M-HS Sinding, G Pacheco, LD Shepherd, AJD Tennyson, S Grosser, E Kay, LJ Nupen, U Ellenberg, DM Houston, AH Reeve, K Johnson, JF Masello, T Stracke, B Mckinlay, PG Borboroglu, D-X Zhang and <b>G Zhang</b>. High-coverage genomes to elucidate the evolution of penguins. GigaScience 8, giz117. (2019)</a></p></li>
<li><p><a href=https://doi.org/10.3390/d11080131">I Khan, E Maldonado, L Silva, D Almeida, WE Johnson, SJ OBrien, <b>G Zhang</b>, ED Jarvis, MTP Gilbert, A Antunes. The Vertebrate TLR Supergene Family Evolved Dynamically by Gene Gain/Loss and Positive Selection Revealing a Host Pathogen Arms Race in Birds. Diversity 8, 131. (2019)</a></p></li>
<li><p><a href="https://doi.org/10.1534/g3.118.200865">KM Kapheim, H Pan, C Li, C Blatti, Iii, BA Harpur, P Ioannidis, BM Jones, CF Kent, L Ruzzante, L Sloofman, E Stolle, RM Waterhouse, A Zayed, <b>G Zhang</b> and WT Wcislo. Draft Genome Assembly and Population Genetics of an Agricultural Pollinator, the Solitary Alkali Bee (Halictidae: Nomia melanderi). G3 Genes|Genomes|Genetics 9, 625-634. (2019)</a></p></li>
<li><p><a href="https://doi.org/10.1007/s11427-019-9481-8">Y He, H Pan, <b>G Zhang</b> and S He. Comparative study on pattern recognition receptors in non-teleost ray-finned fishes and their evolutionary significance in primitive vertebrates. Science China Life Sciences 62, 566-578. (2019)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.1816086116">W Gao, Y-B Sun, W-W Zhou, Z-J Xiong, L Chen, H Li, T-T Fu, K Xu, W Xu, L Ma, Y-J Chen, X-Y Xiang, L Zhou, T Zeng, S Zhang, J-Q Jin, H-M Chen, <b>G Zhang</b>, DM Hillis, X Ji, Y-P Zhang and J Che. Genomic and transcriptomic investigations of the evolutionary transition from oviparity to viviparity. Proceedings of the National Academy of Sciences 116, 3646-3655. (2019)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.cub.2018.12.008">S Feng, Q Fang, R Barnett, C Li, S Han, M Kuhlwilm, L Zhou, H Pan, Y Deng, G Chen, A Gamauf, F Woog, R Prys-Jones, T Marques-Bonet, MTP Gilbert and <b>G Zhang</b>. The Genomic Footprints of the Fall and Recovery of the Crested Ibis. Current Biology 29, 340-349.e347. (2019)</a></p></li>
<li><p><a href="https://doi.org/10.1093/gigascience/giy162">M Farre, Q Li, Y Zhou, J Damas, LG Chemnick, J Kim, OA Ryder, J Ma, <b>G Zhang</b>, DM Larkin and HA Lewin. A near-chromosome-scale genome assembly of the gemsbok (Oryx gazella): an iconic antelope of the Kalahari desert. GigaScience 8, giy162. (2019)</a></p></li>
<li><p><a href="https://doi.org/10.1101/gr.239863.118">M Farre, J Kim, AA Proskuryakova, Y Zhang, AI Kulemzina, Q Li, Y Zhou, Y Xiong, JL Johnson, PL Perelman, WE Johnson, WC Warren, AV Kukekova, <b>G Zhang</b>, SJ O'brien, OA Ryder, AS Graphodatsky, J Ma, HA Lewin and DM Larkin. Evolution of gene regulation in ruminants differs between evolutionary breakpoint regions and homologous synteny blocks. Genome research 29, 576-589. (2019)</a></p></li>
<li><p><a href="https://doi.org/10.24272/j.issn.2095-8137.2019.063">Y Fan, M Ye, J Y Zhang, L Xu, D Yu, T Gu, Y Yao, J Chen, L Lv, P Zheng, D Wu, <b>G Zhang</b> and Y Yao. Chromosomal level assembly and population sequencing of the Chinese tree shrew genome. Zoological Research 40, 506. (2019)</a></p></li>
<li><p><a href="https://doi.org/10.1093/molbev/msz017">TL Cole, DT Ksepka, KJ Mitchell, AJD Tennyson, DB Thomas, H Pan, <b>G Zhang</b>, NJ Rawlence, JR Wood, P Bover, JL Bouzat, A Cooper, SR Fiddaman, T Hart, G Miller, PG Ryan, LD Shepherd, JM Wilmshurst and JM Waters. Mitogenomes Uncover Extinct Penguin Taxa and Reveal Island Formation as a Key Driver of Speciation. Molecular Biology and Evolution 36, 784-797. (2019)</a></p></li>
<li><p><a href="https://doi.org/10.1126/science.aav6202">L Chen, Q Qiu, Y Jiang, K Wang, Z Lin, Z Li, F Bibi, Y Yang, J Wang, W Nie, W Su, G Liu, Q Li, W Fu, X Pan, C Liu, J Yang, C Zhang, Y Yin, Y Wang, Y Zhao, C Zhang, Z Wang, Y Qin, W Liu, B Wang, Y Ren, R Zhang, Y Zeng, RR Da Fonseca, B Wei, R Li, W Wan, R Zhao, W Zhu, Y Wang, S Duan, Y Gao, YE Zhang, C Chen, C Hvilsom, CW Epps, LG Chemnick, Y Dong, S Mirarab, HR Siegismund, OA Ryder, MTP Gilbert, HA Lewin, <b>G Zhang</b>, R Heller and W Wang. Large-scale ruminant genome sequencing provides insights into their evolution and distinct traits. Science 364, eaav6202. (2019)</a></p></li>
<li><p><a href="https://doi.org/10.1093/gbe/evz111">R Borges, J Fonseca, C Gomes, WE Johnson, SJ Obrien, <b>G Zhang</b>, MTP Gilbert, ED Jarvis and A Antunes. Avian Binocularity and Adaptation to Nocturnal Environments: Genomic Insights from a Highly Derived Visual Phenotype. Genome Biology and Evolution 11, 2244-2255. (2019)</a></p></li>

<h2>-- 2018年研究成果 --</h2>
<li><p><a href="https://doi.org/10.1093/nsr/nwy058"><b>G Zhang</b>. The first AsiaEvo conference, connecting Asian evolutionary biologists to the world. National Science Review 5, 614-616. (2018)</a></p></li>
<li><p><a href="https://doi.org/10.1186/s13059-018-1585-z"><b>G Zhang</b>. The bird's-eye view on chromosome evolution. Genome Biology 19, 201. (2018)</a></p></li>
<li><p><a href="https://doi.org/10.1093/gigascience/gix130">C Zhang, L Chen, Y Zhou, K Wang, LG Chemnick, OA Ryder, W Wang, <b>G Zhang</b> and Q Qiu. Draft genome of the milu (Elaphurus davidianus). GigaScience 7, gix130. (2018)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41559-018-0476-8">ML Zepeda Mendoza, Z Xiong, M Escalera-Zamudio, AK Runge, J Theze, D Streicker, HK Frank, E Loza-Rubio, S Liu, OA Ryder, JA Samaniego Castruita, A Katzourakis, G Pacheco, B Taboada, U Lober, OG Pybus, Y Li, E Rojas-Anaya, K Bohmann, A Carmona Baez, CF Arias, S Liu, AD Greenwood, MF Bertelsen, NE White, M Bunce, <b>G Zhang</b>, T Sicheritz-Ponten and MPT Gilbert. Hologenomic adaptations underlying the evolution of sanguivory in the common vampire bat. Nature Ecology & Evolution 2, 659-668. (2018)</a></p></li>
<li><p><a href="https://doi.org/10.1093/molbev/msy185">RL Rogers, L Zhou, C Chu, R Marquez, A Corl, T Linderoth, L Freeborn, MD Macmanes, Z Xiong, J Zheng, C Guo, X Xun, MR Kronforst, K Summers, Y Wu, H Yang, CL Richards-Zawacki, <b>G Zhang</b> and R Nielsen. Genomic Takeover by Transposable Elements in the Strawberry Poison Frog. Molecular Biology and Evolution 35, 2913-2927. (2018)</a></p></li>
<li><p><a href="https://pubmed.ncbi.nlm.nih.gov/29925783/">H Rando, M Farre, M Robson, N Won, J Johnson, R Buch, E Bastounes, X Xiang, S Feng, S Liu, Z Xiong, J Kim, <b>G Zhang</b>, L Trut, D Larkin, A Kukekova. Construction of Red Fox Chromosomal Fragments from the Short-Read Genome Assembly. Genes 9, 308. (2018).</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41559-018-0689-x">B Qiu, RS Larsen, N-C Chang, J Wang, JJ Boomsma and <b>G Zhang</b>. Towards reconstructing the ancestral brain gene-network regulating caste differentiation in ants. Nature Ecology & Evolution 2, 1782-1791. (2018)</a></p></li>
<li><p><a href="https://doi.org/10.1093/gigascience/giy101">C Liu, Y Zhang, Y Ren, H Wang, S Li, F Jiang, L Yin, X Qiao, <b>G Zhang</b>, W Qian, B Liu and W Fan. The genome of the golden apple snail Pomacea canaliculata provides insight into stress tolerance and invasive adaptation. GigaScience 7, giy101. (2018)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.1720115115">HA Lewin, GE Robinson, WJ Kress, WJ Baker, J Coddington, KA Crandall, R Durbin, SV Edwards, F Forest, MTP Gilbert, MM Goldstein, IV Grigoriev, KJ Hackett, D Haussler, ED Jarvis, WE Johnson, A Patrinos, S Richards, JC Castilla-Rubio, M-A Van Sluys, PS Soltis, X Xu, H Yang and <b>G Zhang</b>. Earth BioGenome Project: Sequencing life for the future of life. Proceedings of the National Academy of Sciences 115, 4325-4333. (2018)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41559-018-0611-6">AV Kukekova, JL Johnson, X Xiang, S Feng, S Liu, HM Rando, AV Kharlamova, Y Herbeck, NA Serdyukova, Z Xiong, V Beklemischeva, K-P Koepfli, RG Gulevich, AV Vladimirova, JP Hekman, PL Perelman, AS Graphodatsky, SJ Obrien, X Wang, AG Clark, GM Acland, LN Trut and <b>G Zhang</b>. Red fox genome assembly identifies genomic regions associated with tame and aggressive behaviours. Nature Ecology & Evolution 2, 1479-1491. (2018)</a></p></li>

<h2>-- 2017年研究成果 --</h2>
<li><p><a href="https://doi.org/10.1093/gigascience/gix106">Y Yang, Y Wang, Y Zhao, X Zhang, R Li, L Chen, <b>G Zhang</b>, Y Jiang, Q Qiu, W Wang, H-J Wei and K Wang. Draft genome of the Marco Polo Sheep (Ovis ammon polii). GigaScience 6, gix106. (2017)</a></p></li>
<li><p><a href="https://doi.org/10.1038/ng.3732">C Shao, B Bao, Z Xie, X Chen, B Li, X Jia, Q Yao, G Orti, W Li, X Li, K Hamre, J Xu, L Wang, F Chen, Y Tian, AM Schreiber, N Wang, F Wei, J Zhang, Z Dong, L Gao, J Gai, T Sakamoto, S Mo, W Chen, Q Shi, H Li, Y Xiu, Y Li, W Xu, Z Shi, <b>G Zhang</b>, DM Power, Q Wang, M Schartl and S Chen. The genome and transcriptome of Japanese flounder provide insights into flatfish asymmetry. Nature Genetics 49, 119-124. (2017)</a></p></li>
<li><p><a href="https://doi.org/10.1038/ncomms14229">R Seki, C Li, Q Fang, S Hayashi, S Egawa, J Hu, L Xu, H Pan, M Kondo, T Sato, H Matsubara, N Kamiyama, K Kitajima, D Saito, Y Liu, MTP Gilbert, Q Zhou, X Xu, T Shiroishi, N Irie, K Tamura and <b>G Zhang</b>. Functional roles of Aves class-specific cis-regulatory elements on macroevolution of bird-specific features. Nature Communications 8, 14229. (2017)</a></p></li>
<li><p><a href="https://doi.org/10.1093/gigascience/gix049">SST Mak, S Gopalakrishnan, C Caroe, C Geng, S Liu, M-HS Sinding, LFK Kuderna, W Zhang, S Fu, FG Vieira, M Germonpre, H Bocherens, S Fedorov, B Petersen, T Sicheritz-Ponten, T Marques-Bonet, <b>G Zhang</b>, H Jiang and MTP Gilbert. Comparative performance of the BGISEQ-500 vs Illumina HiSeq2500 sequencing platforms for palaeogenomic sequencing. GigaScience 6, gix049. (2017)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41559-017-0318-0">H Hu, M Uesaka, S Guo, K Shimai, T-M Lu, F Li, S Fujimoto, M Ishikawa, S Liu, Y Sasagawa, <b>G Zhang</b>, S Kuratani, J-K Yu, TG Kusakabe, P Khaitovich, N Irie and EC The. Constrained vertebrate evolution by pleiotropic genes. Nature Ecology & Evolution 1, 1722-1730. (2017)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.1702930114">AD Hargreaves, L Zhou, J Christensen, F Marletaz, S Liu, F Li, PG Jansen, E Spiga, MT Hansen, SVH Pedersen, S Biswas, K Serikawa, BA Fox, WR Taylor, JF Mulley, <b>G Zhang</b>, RS Heller and PWH Holland. Genome sequence of a diabetes-prone rodent reveals a mutation hotspot around the ParaHox gene cluster. Proceedings of the National Academy of Sciences 114, 7677-7682. (2017)</a></p></li>
<li><p><a href="https://doi.org/10.1093/gigascience/gix041">J Gao, Q Li, Z Wang, Y Zhou, P Martelli, F Li, Z Xiong, J Wang, H Yang and <b>G Zhang</b>. Sequencing, de novo assembling, and annotating the genome of the endangered Chinese crocodile lizard Shinisaurus crocodilurus. GigaScience 6, gix041. (2017)</a></p></li>
<li><p><a href="https://myrmecologicalnews.org/cms/index.php?option=com_download&view=download&filename=volume25/mn25_61-66_printable.pdf&format=raw">JJ Boomsma, SG Brady, RR Dunn, J Gadau, J Heinze, L Keller, CS Moreau, NJ Sanders, L Schrader 
TR Schultz, L Sundstrom, PS Ward, W Wcislo and <b>G Zhang</b>. The global ant genomics Alliance. Myrmecological News25, 61-66. (2017)</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41467-017-01800-0">A Avalos, H Pan, C Li, JP Acevedo-Gonzalez, G Rendon, CJ Fields, PJ Brown, T Giray, GE Robinson, ME Hudson and <b>G Zhang</b>. A soft selective sweep during rapid evolution of gentle behaviour in an Africanized honeybee. Nature Communications 8, 1550. (2017)</a></p></li>

<h2>-- 2016年研究成果 --</h2>
<li><p><a href="https://doi.org/10.1038/ncomms13107">W Yin, Z Wang, Q Li, J Lian, Y Zhou, B Lu, L Jin, P Qiu, P Zhang, W Zhu, B Wen, Y Huang, Z Lin, B Qiu, X Su, H Yang, <b>G Zhang</b>, G Yan and Q Zhou. Evolutionary trajectories of snake genes and genomes revealed by comparative analyses of five-pacer viper. Nature Communications 7, 13107. (2016)</a></p></li>
<li><p><a href="https://doi.org/10.1186/s13742-016-0151-4">Z Xiong, F Li, Q Li, L Zhou, T Gamble, J Zheng, L Kui, C Li, S Li, H Yang and <b>G Zhang</b>. Draft genome of the leopard gecko, Eublepharis macularius. GigaScience 5, s13742-13016-10151-13744. (2016)</a></p></li>
<li><p><a href="https://doi.org/10.1186/s13742-016-0143-4">Z Wang, J Lian, Q Li, P Zhang, Y Zhou, X Zhan and <b>G Zhang</b>. RES-Scanner: a software package for genome-wide identification of RNA-editing sites. GigaScience 5, s13742-13016-10143-13744. (2016)</a></p></li>
<li><p><a href="https://doi.org/10.1186/s13742-016-0144-3">H Pan, H Yu, V Ravi, C Li, AP Lee, MM Lian, B-H Tay, S Brenner, J Wang, H Yang, <b>G Zhang</b> and B Venkatesh. The genome of the largest bony fish, ocean sunfish (Mola mola), provides insights into its fast growth rate. GigaScience 5, s13742-13016-10144-13743. (2016)</a></p></li>
<li><p><a href="https://doi.org/10.1038/ncomms12233">S Nygaard, H Hu, C Li, M Schiott, Z Chen, Z Yang, Q Xie, C Ma, Y Deng, RB Dikow, C Rabeling, DR Nash, WT Wcislo, SG Brady, TR Schultz, <b>G Zhang</b> and JJ Boomsma. Reciprocal genomic evolution in the ant-fungus agricultural symbiosis. Nature Communications 7, 12233. (2016)</a></p></li>
<li><p><a href="https://doi.org/10.1186/s12864-016-2681-7">JP Machado, WE Johnson, MTP Gilbert, <b>G Zhang</b>, ED Jarvis, SJ Obrien and A Antunes. Bone-associated gene evolution and the origin of flight in birds. BMC Genomics 17, 371. (2016)</a></p></li>
<li><p><a href="https://doi.org/10.1093/gbe/evw041">D Almeida, E Maldonado, I Khan, L Silva, MTP Gilbert, <b>G Zhang</b>, ED Jarvis, SJ Obrien, WE Johnson and A Antunes. Whole-Genome Identification, Phylogeny, and Evolution of the Cytochrome P450 Family 2 (CYP2) Subfamilies in Birds. Genome Biology and Evolution 8, 1115-1131. (2016)</a></p></li>

<h2>-- 2015年研究成果 --</h2>
<li><p><a href="https://doi.org/10.1186/s13742-015-0062-9">J Zhang, C Li, Q Zhou and <b>G Zhang</b>. Improving the ostrich genome assembly using optical mapping data. GigaScience 4, s13742-13015-10062-13749. (2015)</a></p></li>
<li><p><a href="https://doi.org/10.1038/522034d">144.	<b>G Zhang</b>. Bird sequencing project takes off. Nature 522, 34-34. (2015)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.molp.2014.12.011">L Yan, X Wang, H Liu, Y Tian, J Lian, R Yang, S Hao, X Wang, S Yang, Q Li, S Qi, L Kui, M Okpekum, X Ma, J Zhang, Z Ding, <b>G Zhang</b>, W Wang, Y Dong and J Sheng. The Genome of Dendrobium officinale Illuminates the Biology of the Important Traditional Chinese Orchid Herb. Molecular Plant 8, 922-934. (2015)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.1501764112">Y Sun, Z Xiong, X Xiang, S Liu, W Zhou, X Tu, L Zhong, L Wang, D Wu, B Zhang, C Zhu, M Yang, H Chen, F Li, L Zhou, S Feng, C Huang, <b>G Zhang</b>, D Irwin, DM Hillis, RW Murphy, H Yang, J Che, J Wang and Y Zhang. Whole-genome sequence of the Tibetan frog Nanorana parkeri and the comparative evolution of tetrapod genomes. Proceedings of the National Academy of Sciences 112, E1257-E1262. (2015)</a></p></li>
<li><p><a href="https://doi.org/10.1371/journal.pone.0074132">J Pereira, WE Johnson, SJ Obrien, ED Jarvis, <b>G Zhang</b>, MTP Gilbert, V Vasconcelos and A Antunes. Evolutionary Genomics and Adaptive Evolution of the Hedgehog Gene Family (Shh, Ihh and Dhh) in Vertebrates. PLOS ONE 9, e74132. (2015)</a></p></li>
<li><p><a href="https://doi.org/10.1098/rstb.2013.0381">M Parks, S Subramanian, C Baroni, MC Salvatore, <b>G Zhang</b>, CD Millar and DM Lambert. Ancient population genomics and the study of evolution. Philosophical Transactions of the Royal Society B: Biological Sciences 370, 20130381. (2015)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.cub.2015.03.047">K Nadachowska-Brzyska, C Li, L Smeds, <b>G Zhang</b> and H Ellegren. Temporal Dynamics of Avian Populations during Pleistocene Revealed by Whole-Genome Sequences. Current Biology 25, 1375-1380. (2015)</a></p></li>
<li><p><a href="https://doi.org/10.1038/ncomms9212">X Li, D Fan, W Zhang, G Liu, L Zhang, L Zhao, X Fang, L Chen, Y Dong, Y Chen, Y Ding, R Zhao, M Feng, Y Zhu, Y Feng, X Jiang, D Zhu, H Xiang, X Feng, S Li, J Wang, <b>G Zhang</b>, MR Kronforst and W Wang. Outbred genome sequencing and CRISPR/Cas9 gene editing in butterflies. Nature Communications 6, 8212. (2015)</a></p></li>
<li><p><a href="https://www.frontiersin.org/articles/10.3389/fgene.2015.00009">J Korb, M Poulsen, H Hu, C Li, JJ Boomsma, <b>G Zhang</b> and J Liebig. A genomic comparison of two termites with different social complexity. Frontiers in Genetics 6, (2015)</a></p></li>
<li><p><a href="https://doi.org/10.1093/molbev/msv155">I Khan, Z Yang, E Maldonado, C Li, <b>G Zhang</b>, MTP Gilbert, ED Jarvis, SJ Obrien, WE Johnson and A Antunes. Olfactory Receptor Subgenomes Linked with Broad Ecological Adaptations in Sauropsida. Molecular Biology and Evolution 32, 2832-2843. (2015)</a></p></li>
<li><p><a href="https://doi.org/10.1126/science.aaa4788">KM Kapheim, H Pan, C Li, SL Salzberg, D Puiu, T Magoc, HM Robertson, ME Hudson, A Venkat, BJ Fischman, A Hernandez, M Yandell, D Ence, C Holt, GD Yocum, WP Kemp, J Bosch, RM Waterhouse, EM Zdobnov, E Stolle, FB Kraus, S Helbing, RFA Moritz, KM Glastad, BG Hunt, MaD Goodisman, F Hauser, CJP Grimmelikhuijzen, DG Pinheiro, FMF Nunes, MPM Soares, ED Tanaka, ZLP Simões, K Hartfelder, JD Evans, SM Barribeau, RM Johnson, JH Massey, BR Southey, M Hasselmann, D Hamacher, M Biewer, CF Kent, A Zayed, C Blatti, S Sinha, JS Johnston, SJ Hanrahan, SD Kocher, J Wang, GE Robinson and <b>G Zhang</b>. Genomic signatures of evolutionary transitions from solitary to group living. Science 348, 1139-1143. (2015)</a></p></li>
<li><p><a href="https://doi.org/10.1186/s13742-014-0038-1">ED Jarvis, S Mirarab, AJ Aberer, B Li, P Houde, C Li, SYW Ho, BC Faircloth, B Nabholz, JT Howard, A Suh, CC Weber, RR Da Fonseca, A Alfaro-Nunez, N Narula, L Liu, D Burt, H Ellegren, SV Edwards, A Stamatakis, DP Mindell, J Cracraft, EL Braun, T Warnow, W Jun, MTP Gilbert, <b>G Zhang</b> and C The Avian Phylogenomics. Phylogenomic analyses data of the avian phylogenomics project. GigaScience 4, s13742-13014-10038-13741. (2015)</a></p></li>
<li><p><a href="https://doi.org/10.1186/s13742-015-0085-2">A Georges, Q Li, J Lian, D Omeally, J Deakin, Z Wang, P Zhang, M Fujita, HR Patel, CE Holleley, Y Zhou, X Zhang, K Matsubara, P Waters, JaM Graves, SD Sarre and <b>G Zhang</b>. High-coverage sequencing and annotated assembly of the genome of the Australian dragon lizard Pogona vitticeps. GigaScience 4, s13742-13015-10085-13742. (2015)</a></p></li>
<li><p><a href="https://doi.org/10.1186/s13059-015-0588-2">L Eory, MTP Gilbert, C Li, B Li, A Archibald, BL Aken, <b>G Zhang</b>, E Jarvis, P Flicek and DW Burt. Avianbase: a community resource for bird genomics. Genome Biology 16, 21. (2015)</a></p></li>
<li><p><a href="https://doi.org/10.1186/s12864-015-1606-1">Y Dong, X Zhang, M Xie, B Arefnezhad, Z Wang, W Wang, S Feng, G Huang, R Guan, W Shen, R Bunch, R Mcculloch, Q Li, B Li, <b>G Zhang</b>, X Xu, JW Kijas, GH Salekdeh, W Wang and Y Jiang. Reference genome of wild goat (capra aegagrus) and sequencing of goat breeds provide insight into genic basis of goat domestication. BMC Genomics 16, 431. (2015)</a></p></li>
<li><p><a href="https://doi.org/10.1186/s13059-015-0837-4">P Dobrynin, S Liu, G Tamazian, Z Xiong, AA Yurchenko, K Krasheninnikova, S Kliver, A Schmidt-Kuntzel, K-P Koepfli, W Johnson, LFK Kuderna, R Garcia-Perez, MD Manuel, R Godinez, A Komissarov, A Makunin, V Brukhin, W Qiu, L Zhou, F Li, J Yi, C Driscoll, A Antunes, TK Oleksyk, E Eizirik, P Perelman, M Roelke, D Wildt, M Diekhans, T Marques-Bonet, L Marker, J Bhak, J Wang, <b>G Zhang</b> and SJ Obrien. Genomic legacy of the African cheetah, Acinonyx jubatus. Genome Biology 16, 277. (2015)</a></p></li>
<li><p><a href="https://doi.org/10.1186/s12864-015-1924-3">R Borges, I Khan, WE Johnson, MTP Gilbert, <b>G Zhang</b>, ED Jarvis, SJ Obrien and A Antunes. Gene loss, adaptive evolution and the co-evolution of plumage coloration genes with opsins in birds. BMC Genomics 16, 751. (2015)</a></p></li>

<h2>-- 2014年研究成果 --</h2>
<li><p><a href="https://doi.org/10.1126/science.1246338">Q Zhou, J Zhang, D Bachtrog, N An, Q Huang, ED Jarvis, MTP Gilbert and <b>G Zhang</b>. Complex evolutionary trajectories of sex chromosomes across bird taxa. Science 346, 1246338. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1126/science.1251385"><b>G Zhang</b>, C Li, Q Li, B Li, DM Larkin, C Lee, JF Storz, A Antunes, MJ Greenwold, RW Meredith, A Odeen, J Cui, Q Zhou, L Xu, H Pan, Z Wang, L Jin, P Zhang, H Hu, W Yang, J Hu, J Xiao, Z Yang, Y Liu, Q Xie, H Yu, J Lian, P Wen, F Zhang, H Li, Y Zeng, Z Xiong, S Liu, L Zhou, Z Huang, N An, J Wang, Q Zheng, Y Xiong, G Wang, B Wang, J Wang, Y Fan, RR Da Fonseca, A Alfaro-Numez, M Schubert, L Orlando, T Mourier, JT Howard, G Ganapathy, A Pfenning, O Whitney, MV Rivas, E Hara, J Smith, M FarreeJ Narayan, G Slavov, MN Romanov, R Borges, JP Machado, I Khan, MS Springer, J Gatesy, FG Hoffmann, JC Opazo, O Hastad, RH Sawyer, H Kim, K-W Kim, HJ Kim, S Cho, N Li, Y Huang, MW Bruford, X Zhan, A Dixon, MF Bertelsen, E Derryberry, W Warren, RK Wilson, S Li, DA Ray, RE Green, SJ Obrien, D Griffin, WE Johnson, D Haussler, OA Ryder, E Willerslev, GR Graves, P Alstrum, J Fjeldsa, DP Mindell, SV Edwards, EL Braun, C Rahbek, DW Burt, P Houde, Y Zhang, H Yang, J Wang, C Avian Genome, ED Jarvis, MTP Gilbert, J Wang, C Ye, S Liang, Z Yan, ML Zepeda, PF Campos, AMV Velazquez, JA Samaniego, M Avila-Arcos, MD Martin, R Barnett, AM Ribeiro, CV Mello, PV Lovell, D Almeida, E Maldonado, J Pereira, K Sunagar, S Philip, MG Dominguez-Bello, M Bunce, D Lambert, RT Brumfield, FH Sheldon, EC Holmes, PP Gardner, TE Steeves, PF Stadler, SW Burge, E Lyons, J Smith, F Mccarthy, F Pitel, D Rhoads and DP Froman. Comparative genomics reveals insights into avian genome evolution and adaptation. Science 346, 1311-1320. (2014)</a></p></li>
<li><p><a href="http://gigadb.org/dataset/101013"><b>G Zhang</b>, B Li, C Li, MTP Gilbert, O Ryder and ED Jarvis. Genomic data of the Ostrich (Struthio camelus australis). GigaScience Database (2014)</a></p></li>
<li><p><a href="http://gigadb.org/dataset/101004"><b>G Zhang</b>, B Li, C Li, MTP Gilbert, CV Mello, ED Jarvis, C Avian Genome and J Wang. Genomic data of the Anna's hummingbird (Calypte anna). GigaScience Database (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1186/2047-217X-3-26"><b>G Zhang</b>, B Li, C Li, MTP Gilbert, ED Jarvis, J Wang and C The Avian Genome. Comparative genomic data of the Avian Phylogenomics Project. GigaScience 3, 2047-2217X-2043-2026. (2014)</a></p></li>
<li><p><a href="http://gigadb.org/dataset/101000"><b>G Zhang</b>, B Li, C Li, MTP Gilbert, E Jarvis and J Wang. The avian phylogenomic project data. GigaScience Database (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1126/science.346.6215.1308"><b>G Zhang</b>, ED Jarvis and MTP Gilbert. A flock of genomes. Science 346, 1308-1309. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1038/ncomms6594">X You, C Bian, Q Zan, X Xu, X Liu, J Chen, J Wang, Y Qiu, W Li, X Zhang, Y Sun, S Chen, W Hong, Y Li, S Cheng, G Fan, C Shi, J Liang, Y Tom Tang, C Yang, Z Ruan, J Bai, C Peng, Q Mu, J Lu, M Fan, S Yang, Z Huang, X Jiang, X Fang, <b>G Zhang</b>, Y Zhang, G Polgar, H Yu, J Li, Z Liu, G Zhang, V Ravi, SL Coon, J Wang, H Yang, B Venkatesh, J Wang and Q Shi. Mudskipper genomes provide insights into the terrestrial adaptation of amphibious fishes. Nature Communications 5, 5594. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1186/s12862-014-0250-8">Z Wang, J Zhang, W Yang, N An, P Zhang, <b>G Zhang</b> and Q Zhou. Temporal genomic evolution of bird sex chromosomes. BMC Evolutionary Biology 14, 250. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1038/ncomms3957">X Wang, X Fang, P Yang, X Jiang, F Jiang, D Zhao, B Li, F Cui, J Wei, C Ma, Y Wang, J He, Y Luo, Z Wang, X Guo, W Guo, X Wang, Y Zhang, M Yang, S Hao, B Chen, Z Ma, D Yu, Z Xiong, Y Zhu, D Fan, L Han, B Wang, Y Chen, J Wang, L Yang, W Zhao, Y Feng, G Chen, J Lian, Q Li, Z Huang, X Yao, N Lv, <b>G Zhang</b>, Y Li, J Wang, J Wang, B Zhu and L Kang. The locust genome provides insight into swarm formation and long-distance flight. Nature Communications 5, 2957. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1038/ncomms4636">N Terrapon, C Li, HM Robertson, L Ji, X Meng, W Booth, Z Chen, CP Childers, KM Glastad, K Gokhale, J Gowin, W Gronenberg, RA Hermansen, H Hu, BG Hunt, AK Huylmans, SMS Khalil, RD Mitchell, MC Munoz-Torres, JA Mustard, H Pan, JT Reese, ME Scharf, F Sun, H Vogel, J Xiao, W Yang, Z Yang, Z Yang, J Zhou, J Zhu, CS Brent, CG Elsik, MaD Goodisman, DA Liberles, RM Roe, EL Vargo, A Vilcinskas, J Wang, E Bornberg-Bauer, J Korb, <b>G Zhang</b> and J Liebig. Molecular traces of alternative social organization in a termite genome. Nature Communications 5, 3636. (2014)</a></p></li>
<li><p><a href="https://genome.cshlp.org/content/24/4/604">C Shao, Q Li, S Chen, P Zhang, J Lian, Q Hu, B Sun, L Jin, S Liu, Z Wang, H Zhao, Z Jin, Z Liang, Y Li, Q Zheng, Y Zhang, J Wang and <b>G Zhang</b>. Epigenetic modification and inheritance in sexual reversal of fish. Genome research 24, 604-615. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.1319718111">M Poulsen, H Hu, C Li, Z Chen, L Xu, S Otani, S Nygaard, T Nobre, S Klaubauf, PM Schindler, F Hauser, H Pan, Z Yang, ASM Sonnenberg, ZW De Beer, Y Zhang, MJ Wingfield, CJP Grimmelikhuijzen, RP De Vries, J Korb, DK Aanen, J Wang, JJ Boomsma and <b>G Zhang</b>. Complementary symbiont contributions to plant decomposition in a fungus-farming termite. Proceedings of the National Academy of Sciences 111, 14500-14505. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1126/science.1256846">AR Pfenning, E Hara, O Whitney, MV Rivas, R Wang, PL Roulhac, JT Howard, M Wirthlin, PV Lovell, G Ganapathy, J Mountcastle, MA Moseley, JW Thompson, EJ Soderblom, A Iriki, M Kato, MTP Gilbert, <b>G Zhang</b>, T Bakken, A Bongaarts, A Bernard, E Lein, CV Mello, AJ Hartemink and ED Jarvis. Convergent transcriptional specializations in the brains of humans and song-learning birds. Science 346, 1256846. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.cub.2014.01.018">Peter r Oxley, L Ji, I Fetter-Pruneda, Sean k Mckenzie, C Li, H Hu, <b>G Zhang</b> and Daniel jC Kronauer. The Genome of the Clonal Raider Ant Cerapachys biroi. Current Biology 24, 451-458. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1126/science.1254390">RW Meredith, <b>G Zhang</b>, MTP Gilbert, ED Jarvis and MS Springer. Evidence for a single loss of mineralized teeth in the common avian ancestor. Science 346, 1254390. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.cell.2014.03.054">S Liu, Eline d Lorenzen, M Fumagalli, B Li, K Harris, Z Xiong, L Zhou, Thorfinn s Korneliussen, M Somel, C Babbitt, G Wray, J Li, W He, Z Wang, W Fu, X Xiang, Claire c Morgan, A Doherty, Mary j Oconnell, James o Mcinerney, Erik w Born, L Dalen, R Dietz, L Orlando, C Sonne, <b>G Zhang</b>, R Nielsen, E Willerslev and J Wang. Population Genomics Reveal Recent Speciation and Rapid Evolutionary Adaptation in Polar Bears. Cell 157, 785-794. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1186/s13059-014-0557-1">S Li, B Li, C Cheng, Z Xiong, Q Liu, J Lai, HV Carey, Q Zhang, H Zheng, S Wei, H Zhang, L Chang, S Liu, S Zhang, B Yu, X Zeng, Y Hou, W Nie, Y Guo, T Chen, J Han, J Wang, J Wang, C Chen, J Liu, PJ Stambrook, M Xu, <b>G Zhang</b>, MTP Gilbert, H Yang, ED Jarvis, J Yu and J Yan. Genomic signatures of near-extinction and rebirth of the crested ibis and other endangered bird species. Genome Biology 15, 557. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1038/ncomms5943">Q Li, Z Wang, J Lian, M Schiott, L Jin, P Zhang, Y Zhang, S Nygaard, Z Peng, Y Zhou, Y Deng, W Zhang, JJ Boomsma and <b>G Zhang</b>. Caste-specific RNA editomes in the leaf-cutting ant Acromyrmex echinatior. Nature Communications 5, 4943. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1186/2047-217X-3-27">C Li, Y Zhang, J Li, L Kong, H Hu, H Pan, L Xu, Y Deng, Q Li, L Jin, H Yu, Y Chen, B Liu, L Yang, S Liu, Y Zhang, Y Lang, J Xia, W He, Q Shi, S Subramanian, CD Millar, S Meader, CM Rands, MK Fujita, MJ Greenwold, TA Castoe, DD Pollock, W Gu, K Nam, H Ellegren, SYW Ho, DW Burt, CP Ponting, ED Jarvis, MTP Gilbert, H Yang, J Wang, DM Lambert, J Wang and <b>G Zhang</b>. Two Antarctic penguin genomes reveal insights into their evolutionary history and molecular changes related to the Antarctic environment. GigaScience 3, 2047-2217X-2043-2027. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1126/science.1253451">ED Jarvis, S Mirarab, AJ Aberer, B Li, P Houde, C Li, SYW Ho, BC Faircloth, B Nabholz, JT Howard, A Suh, CC Weber, RR Da Fonseca, J Li, F Zhang, H Li, L Zhou, N Narula, L Liu, G Ganapathy, B Boussau, MS Bayzid, V Zavidovych, S Subramanian, T Gabaldon, S Capella-Gutierrez, J Huerta-Cepas, B Rekepalli, K Munch, M Schierup, B Lindow, WC Warren, D Ray, RE Green, MW Bruford, X Zhan, A Dixon, S Li, N Li, Y Huang, EP Derryberry, MF Bertelsen, FH Sheldon, RT Brumfield, CV Mello, PV Lovell, M Wirthlin, MPC Schneider, F Prosdocimi, JA Samaniego, AMV Velazquez, A Alfaro-Nunez, PF Campos, B Petersen, T Sicheritz-Ponten, A Pas, T Bailey, P Scofield, M Bunce, DM Lambert, Q Zhou, P Perelman, AC Driskell, B Shapiro, Z Xiong, Y Zeng, S Liu, Z Li, B Liu, K Wu, J Xiao, X Yinqi, Q Zheng, Y Zhang, H Yang, J Wang, L Smeds, FE Rheindt, M Braun, J Fjeldsa, L Orlando, FK Barker, KA Jonsson, W Johnson, K-P Koepfli, S Obrien, D Haussler, OA Ryder, C Rahbek, E Willerslev, GR Graves, TC Glenn, J Mccormack, D Burt, H Ellegren, P Alstrum, SV Edwards, A Stamatakis, DP Mindell, J Cracraft, EL Braun, T Warnow, W Jun, MTP Gilbert and <b>G Zhang</b>. Whole-genome analyses resolve early branches in the tree of life of modern birds. Science 346, 1320-1331. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1534/g3.114.011262">NJ Hudson, ML Baker, NS Hart, JW Wynne, Q Gu, Z Huang, <b>G Zhang</b>, AB Ingham, L Wang and A Reverter. Sensory Rewiring in an Echolocator: Genome-Wide Modification of Retinogenic and Auditory Genes in the Bat Myotis davidii. G3 Genes|Genomes|Genetics 4, 1825-1835. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1186/s12862-014-0249-1">MJ Greenwold, W Bao, ED Jarvis, H Hu, C Li, MTP Gilbert, <b>G Zhang</b> and RH Sawyer. Dynamic evolution of the alpha and beta keratins has accompanied integument diversification and the adaptation of birds into novel lifestyles. BMC Evolutionary Biology 14, 249. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1126/science.1254449">RE Green, EL Braun, J Armstrong, D Earl, N Nguyen, G Hickey, MW Vandewege, JA St. John, S Capella-Gutierrez, TA Castoe, C Kern, MK Fujita, JC Opazo, J Jurka, KK Kojima, J Caballero, RM Hubley, AF Smit, RN Platt, CA Lavoie, MP Ramakodi, JW Finger, A Suh, SR Isberg, L Miles, AY Chong, W Jaratlerdsiri, J Gongora, C Moran, A Iriarte, J Mccormack, SC Burgess, SV Edwards, E Lyons, C Williams, M Breen, JT Howard, CR Gresham, DG Peterson, J Schmitz, DD Pollock, D Haussler, EW Triplett, <b>G Zhang</b>, N Irie, ED Jarvis, CA Brochu, CJ Schmidt, FM Mccarthy, BC Faircloth, FG Hoffmann, TC Glenn, T Gabaldon, B Paten and DA Ray. Three crocodilian genomes reveal ancestral patterns of evolution among archosaurs. Science 346, 1254449. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1186/2047-217X-3-11">G Ganapathy, JT Howard, JM Ward, J Li, B Li, Y Li, Y Xiong, Y Zhang, S Zhou, DC Schwartz, M Schatz, R Aboukhalil, O Fedrigo, L Bukovnik, T Wang, G Wray, I Rasolonjatovo, R Winer, JR Knight, S Koren, WC Warren, <b>G Zhang</b>, AM Phillippy and ED Jarvis. High-coverage sequencing and annotated assemblies of the budgerigar genome. GigaScience 3, 2047-2217X-2043-2011. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.celrep.2014.07.030">X Fang, I Seim, Z Huang, Maxim v Gerashchenko, Z Xiong, Anton a Turanov, Y Zhu, Alexei v Lobanov, D Fan, Sun h Yim, X Yao, S Ma, L Yang, S-G Lee, Eun b Kim, Roderick t Bronson, R Šumbera, R Buffenstein, X Zhou, A Krogh, Thomas j Park, <b>G Zhang</b>, J Wang and Vadim n Gladyshev. Adaptations to a Subterranean Environment and Longevity Revealed by the Analysis of Mole Rat Genomes. Cell Reports 8, 1354-1364. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1186/s13059-014-0539-3">J Cui, W Zhao, Z Huang, ED Jarvis, MTP Gilbert, PJ Walker, EC Holmes and <b>G Zhang</b>. Low frequency of paleoviral infiltration across the avian phylogeny. Genome Biology 15, 539. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1038/ng.2890">S Chen, <b>G Zhang</b>, C Shao, Q Huang, G Liu, P Zhang, W Song, N An, D Chalopin, J-N Volff, Y Hong, Q Li, Z Sha, H Zhou, M Xie, Q Yu, Y Liu, H Xiang, N Wang, K Wu, C Yang, Q Zhou, X Liao, L Yang, Q Hu, J Zhang, L Meng, L Jin, Y Tian, J Lian, J Yang, G Miao, S Liu, Z Liang, F Yan, Y Li, B Sun, H Zhang, J Zhang, Y Zhu, M Du, Y Zhao, M Schartl, Q Tang and J Wang. Whole-genome sequence of a flatfish provides insights into ZW sex chromosome evolution and adaptation to a benthic lifestyle. Nature Genetics 46, 253-260. (2014)</a></p></li>
<li><p><a href="https://doi.org/10.1186/2047-217X-3-24">L Chen, L Tang, H Xiang, L Jin, Q Li, Y Dong, W Wang and <b>G Zhang</b>. Advances in genome editing technology and its promising application in evolutionary and ecological studies. GigaScience 3, 2047-2217X-2043-2024. (2014)</a></p></li>

<h2>-- 2013年研究成果 --</h2>
<li><p><a href="https://doi.org/10.1126/science.1230835"><b>G Zhang</b>, C Cowled, Z Shi, Z Huang, KA Bishop-Lilly, X Fang, JW Wynne, Z Xiong, ML Baker, W Zhao, M Tachedjian, Y Zhu, P Zhou, X Jiang, J Ng, L Yang, L Wu, J Xiao, Y Feng, Y Chen, X Sun, Y Zhang, GA Marsh, G Crameri, CC Broder, KG Frey, L-F Wang and J Wang. Comparative Analysis of Bat Genomes Provides Insight into the Evolution of Flight and Immunity. Science 339, 456-460. (2013)</a></p></li>
<li><p><a href="https://doi.org/10.1186/1471-2164-14-646">H Xiang, X Li, F Dai, X Xu, A Tan, L Chen, <b>G Zhang</b>, Y Ding, Q Li, J Lian, A Willden, Q Guo, Q Xia, J Wang and W Wang. Comparative methylomics between domesticated and wild silkworms implies possible epigenetic influences on silkworm domestication. BMC Genomics 14, 646. (2013)</a></p></li>
<li><p><a href="https://doi.org/10.1038/ng.2615">Z Wang, J Pascual-Anaya, A Zadissa, W Li, Y Niimura, Z Huang, C Li, S White, Z Xiong, D Fang, B Wang, Y Ming, Y Chen, Y Zheng, S Kuraku, M Pignatelli, J Herrero, K Beal, M Nozawa, Q Li, J Wang, H Zhang, L Yu, S Shigenobu, J Wang, J Liu, P Flicek, S Searle, J Wang, S Kuratani, Y Yin, B Aken, <b>G Zhang</b> and N Irie. The draft genomes of soft-shell turtle and green sea turtle yield insights into the development and evolution of the turtle-specific body plan. Nature Genetics 45, 701-706. (2013)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.fsi.2013.06.017">TT Tan, M Chen, JA Harikrishna, N Khairuddin, MI Mohd Shamsudin, <b>G Zhang</b> and S Bhassu. Deep parallel sequencing reveals conserved and novel miRNAs in gill and hepatopancreas of giant freshwater prawn. Fish & Shellfish Immunology 35, 1061-1069. (2013)</a></p></li>
<li><p><a href="https://doi.org/10.1126/science.1230422">MD Shapiro, Z Kronenberg, C Li, ET Domyan, H Pan, M Campbell, H Tan, CD Huff, H Hu, AI Vickrey, SCA Nielsen, SA Stringham, H Hu, E Willerslev, MTP Gilbert, M Yandell, <b>G Zhang</b> and J Wang. Genomic Diversity and Evolution of the Head Crest in the Rock Pigeon. Science 339, 1063-1067. (2013)</a></p></li>
<li><p><a href="https://doi.org/10.1038/ncomms3212">I Seim, X Fang, Z Xiong, AV Lobanov, Z Huang, S Ma, Y Feng, AA Turanov, Y Zhu, TL Lenz, MV Gerashchenko, D Fan, S Hee Yim, X Yao, D Jordan, Y Xiong, Y Ma, AN Lyapunov, G Chen, OI Kulakova, Y Sun, S-G Lee, RT Bronson, AA Moskalev, SR Sunyaev, <b>G Zhang</b>, A Krogh, J Wang and VN Gladyshev. Genome analysis reveals insights into physiology and longevity of the Brandt's bat Myotis brandtii. Nature Communications 4, 2212. (2013)</a></p></li>
<li><p><a href="https://doi.org/10.1186/gb-2013-14-12-r142">SD Kocher, C Li, W Yang, H Tan, SV Yi, X Yang, HE Hoekstra, <b>G Zhang</b>, NE Pierce and DW Yu. The draft genome of a socially polymorphic halictid bee, Lasioglossum albipes. Genome Biology 14, R142. (2013)</a></p></li>
<li><p><a href=" https://doi.org/10.1111/mec.12201">TL Parchman, Z Gompert, MJ Braun, RT Brumfield, DB Mcdonald, JaC Uy, <b>G Zhang</b>, ED Jarvis, BA Schlinger and CA Buerkle. The genomic consequences of adaptive divergence and reproductive isolation between species of manakins. Molecular Ecology 22, 3304-3317. (2013)</a></p></li>
<li><p><a href="https://doi.org/10.1186/1471-2164-14-186">G Pan, J Xu, T Li, Q Xia, S-L Liu, <b>G Zhang</b>, S Li, C Li, H Liu, L Yang, T Liu, X Zhang, Z Wu, W Fan, X Dang, H Xiang, M Tao, Y Li, J Hu, Z Li, L Lin, J Luo, L Geng, L Wang, M Long, Y Wan, N He, Z Zhang, C Lu, PJ Keeling, J Wang, Z Xiang and Z Zhou. Comparative genomics of parasitic silkworm microsporidia reveal an association between genome expansion and host adaptation. BMC Genomics 14, 186. (2013)</a></p></li>
<li><p><a href="https://doi.org/10.1038/nature12323">L Orlando, A Ginolhac, <b>G Zhang</b>, D Froese, A Albrechtsen, M Stiller, M Schubert, E Cappellini, B Petersen, I Moltke, PLF Johnson, M Fumagalli, JT Vilstrup, M Raghavan, T Korneliussen, A-S Malaspinas, J Vogt, D Szklarczyk, CD Kelstrup, J Vinther, A Dolocan, J Stenderup, AMV Velazquez, J Cahill, M Rasmussen, X Wang, J Min, GD Zazula, A Seguin-Orlando, C Mortensen, K Magnussen, JF Thompson, J Weinstock, K Gregersen, KH Roed, V Eisenmann, CJ Rubin, DC Miller, DF Antczak, MF Bertelsen, S Brunak, KaS Al-Rasheid, O Ryder, L Andersson, J Mundy, A Krogh, MTP Gilbert, K Kjær, T Sicheritz-Ponten, LJ Jensen, JV Olsen, M Hofreiter, R Nielsen, B Shapiro, J Wang and E Willerslev. Recalibrating Equus evolution using the genome sequence of an early Middle Pleistocene horse. Nature 499, 74-78. (2013)</a></p></li>
<li><p><a href="https://doi.org/10.1371/journal.pone.0060839">MI Mohd-Shamsudin, Y Kang, Z Lili, TT Tan, QB Kwong, H Liu, <b>G Zhang</b>, RY Othman and S Bhassu. In-Depth Tanscriptomic Analysis on Giant Freshwater Prawns. PLOS ONE 8, e60839. (2013)
<li><p><a href="https://doi.org/10.1186/gb-2013-14-12-r142">SD Kocher, C Li, W Yang, H Tan, SV Yi, X Yang, HE Hoekstra, <b>G Zhang</b>, NE Pierce and DW Yu. The draft genome of a socially polymorphic halictid bee, Lasioglossum albipes. Genome Biology 14, R142. (2013)</a></p></li>
<li><p><a href="https://doi.org/10.1073/pnas.1220349110">J Kim, DM Larkin, Q Cai, Y Zhang, R-L Ge, L Auvil, B Capitanu, <b>G Zhang</b>, HA Lewin and J Ma. Reference-assisted chromosome assembly. Proceedings of the National Academy of Sciences 110, 1785-1790. (2013)</a></p></li>
<li><p><a href="https://doi.org/10.1093/jhered/est050">KC I. The i5K Initiative: Advancing Arthropod Genomics for Knowledge, Human Health, Agriculture, and the Environment. Journal of Heredity 104, 595-600. (2013)</a></p></li>
<li><p><a href="https://doi.org/10.1038/ncomms2860">R-L Ge, Q Cai, Y-Y Shen, A San, L Ma, Y Zhang, X Yi, Y Chen, L Yang, Y Huang, R He, Y Hui, M Hao, Y Li, B Wang, X Ou, J Xu, Y Zhang, K Wu, C Geng, W Zhou, T Zhou, DM Irwin, Y Yang, L Ying, H Bao, J Kim, DM Larkin, J Ma, HA Lewin, J Xing, RN Platt, DA Ray, L Auvil, B Capitanu, X Zhang, <b>G Zhang</b>, RW Murphy, J Wang, Y-P Zhang and J Wang. Draft genome sequence of the Tibetan antelope. Nature Communications 4, 1858. (2013)</a></p></li>
<li><p><a href="https://doi.org/10.1038/ncomms2416">Y Fan, Z Huang, C Cao, C Chen, Y Chen, D Fan, J He, H Hou, L Hu, X Hu, X Jiang, R Lai, Y Lang, B Liang, S Liao, D Mu, Y Ma, Y Niu, X Sun, J Xia, J Xiao, Z Xiong, L Xu, L Yang, Y Zhang, W Zhao, X Zhao, Y Zheng, J Zhou, Y Zhu, <b>G Zhang</b>, J Wang and Y Yao. Genome of the Chinese tree shrew. Nature Communications 4, 1426. (2013)</a></p></li>
<li><p><a href="https://doi.org/10.1038/nbt.2478">Y Dong, M Xie, Y Jiang, N Xiao, X Du, W Zhang, G Tosser-Klopp, J Wang, S Yang, J Liang, W Chen, J Chen, P Zeng, Y Hou, C Bian, S Pan, Y Li, X Liu, W Wang, B Servin, B Sayre, B Zhu, D Sweeney, R Moore, W Nie, Y Shen, R Zhao, <b>G Zhang</b>, J Li, T Faraut, J Womack, Y Zhang, J Kijas, N Cockett, X Xu, S Zhao, J Wang and W Wang. Sequencing and automated whole-genome optical mapping of the genome of a domestic goat (Capra hircus). Nature Biotechnology 31, 135-141. (2013)</a></p></li>
<li><p><a href="https://doi.org/10.1534/g3.113.008185">JC Chiu, X Jiang, L Zhao, CA Hamm, JM Cridland, P Saelao, KA Hamby, EK Lee, RS Kwok, <b>G Zhang</b>, FG Zalom, VM Walton and DJ Begun. Genome of Drosophila suzukii, the Spotted Wing Drosophila. G3 Genes|Genomes|Genetics 3, 2257-2271. (2013)</a></p></li>
<li><p><a href="https://doi.org/10.1186/2047-217X-2-10">KR Bradnam, JN Fass, A Alexandrov, P Baranay, M Bechner, I Birol, S Boisvert, JA Chapman, G Chapuis, R Chikhi, H Chitsaz, W-C Chou, J Corbeil, C Del Fabbro, TR Docking, R Durbin, D Earl, S Emrich, P Fedotov, NA Fonseca, G Ganapathy, RA Gibbs, S Gnerre, E Godzaridis, S Goldstein, M Haimel, G Hall, D Haussler, JB Hiatt, IY Ho, J Howard, M Hunt, SD Jackman, DB Jaffe, ED Jarvis, H Jiang, S Kazakov, PJ Kersey, JO Kitzman, JR Knight, S Koren, T-W Lam, D Lavenier, F Laviolette, Y Li, Z Li, B Liu, Y Liu, R Luo, I Maccallum, MD Macmanes, N Maillet, S Melnikov, D Naquin, Z Ning, TD Otto, B Paten, OS Paulo, AM Phillippy, F Pina-Martins, M Place, D Przybylski, X Qin, C Qu, FJ Ribeiro, S Richards, DS Rokhsar, JG Ruby, S Scalabrin, MC Schatz, DC Schwartz, A Sergushichev, T Sharpe, TI Shaw, J Shendure, Y Shi, JT Simpson, H Song, F Tsarev, F Vezzi, R Vicedomini, BM Vieira, J Wang, KC Worley, S Yin, S-M Yiu, J Yuan, <b>G Zhang</b>, H Zhang, S Zhou and IF Korf. Assemblathon 2: evaluating de novo methods of genome assembly in three vertebrate species. GigaScience 2, 2047-2217X-2042-2010. (2013)</a></p></li>

<h2>-- 2012年研究成果 --</h2>
<li><p><a href="https://doi.org/10.1186/1471-2164-13-109">Q Zhou, H Zhu, Q Huang, L Zhao, <b>G Zhang</b>, SW Roy, B Vicoso, Z Xuan, J Ruan, Y Zhang, R Zhao, C Ye, X Zhang, J Wang, W Wang and D Bachtrog. Deciphering neo-sex and B chromosome evolution by the draft genome of Drosophila albomicans. BMC Genomics 13, 109. (2012)</a></p></li>
<li><p><a href="https://doi.org/10.1038/nature11413"><b>G Zhang</b>, X Fang, X Guo, L Li, R Luo, F Xu, P Yang, L Zhang, X Wang, H Qi, Z Xiong, H Que, Y Xie, PWH Holland, J Paps, Y Zhu, F Wu, Y Chen, J Wang, C Peng, J Meng, L Yang, J Liu, B Wen, N Zhang, Z Huang, Q Zhu, Y Feng, A Mount, D Hedgecock, Z Xu, Y Liu, T Domazet-Lošo, Y Du, X Sun, S Zhang, B Liu, P Cheng, X Jiang, J Li, D Fan, W Wang, W Fu, T Wang, B Wang, J Zhang, Z Peng, Y Li, N Li, J Wang, M Chen, Y He, F Tan, X Song, Q Zheng, R Huang, H Yang, X Du, L Chen, M Yang, PM Gaffney, S Wang, L Luo, Z She, Y Ming, W Huang, S Zhang, B Huang, Y Zhang, T Qu, P Ni, G Miao, J Wang, Q Wang, CEW Steinberg, H Wang, N Li, L Qian, <b>G Zhang</b>, Y Li, H Yang, X Liu, J Wang, Y Yin and J Wang. The oyster genome reveals stress adaptation and complexity of shell formation. Nature 490, 49-54. (2012)</a></p></li>
<li><p><a href="https://doi.org/10.1038/ng.1065">ND Young, AR Jex, B Li, S Liu, L Yang, Z Xiong, Y Li, C Cantacessi, RS Hall, X Xu, F Chen, X Wu, A Zerlotini, G Oliveira, A Hofmann, <b>G Zhang</b>, X Fang, Y Kang, BE Campbell, A Loukas, S Ranganathan, D Rollinson, G Rinaldi, PJ Brindley, H Yang, J Wang, J Wang and RB Gasser. Whole-genome sequence of Schistosoma haematobium. Nature Genetics 44, 221-225. (2012)</a></p></li>
<li><p><a href="https://doi.org/10.1038/nbt.2050">X Xu, X Liu, S Ge, JD Jensen, F Hu, X Li, Y Dong, RN Gutenkunst, L Fang, L Huang, J Li, W He, <b>G Zhang</b>, X Zheng, F Zhang, Y Li, C Yu, K Kristiansen, X Zhang, J Wang, M Wright, S Mccouch, R Nielsen, J Wang and W Wang. Resequencing 50 accessions of cultivated and wild rice yields markers for identifying agronomically important genes. Nature Biotechnology 30, 105-111. (2012)</a></p></li>
<li><p><a href="https://doi.org/10.1038/ng.2343">Q Qiu, <b>G Zhang</b>, T Ma, W Qian, J Wang, Z Ye, C Cao, Q Hu, J Kim, DM Larkin, L Auvil, B Capitanu, J Ma, HA Lewin, X Qian, Y Lang, R Zhou, L Wang, K Wang, J Xia, S Liao, S Pan, X Lu, H Hou, Y Wang, X Zang, Y Yin, H Ma, J Zhang, Z Wang, Y Zhang, D Zhang, T Yonezawa, M Hasegawa, Y Zhong, W Liu, Y Zhang, Z Huang, S Zhang, R Long, H Yang, J Wang, JA Lenstra, DN Cooper, Y Wu, J Wang, P Shi, J Wang and J Liu. The yak genome and adaptation to life at high altitude. Nature Genetics 44, 946-949. (2012)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.febslet.2012.01.048">Y Li, JH Robins, J Ye, Z Huang, Q Wen and <b>G Zhang</b>. Adaptive diversity of innate immune receptor family short pentraxins in Murinae. FEBS Letters 586, 798-803. (2012)</a></p></li>
<li><p><a href="https://doi.org/10.1186/1471-2164-13-300">X Li, J Zhu, F Hu, S Ge, M Ye, H Xiang, <b>G Zhang</b>, X Zheng, H Zhang, S Zhang, Q Li, R Luo, C Yu, J Yu, J Sun, X Zou, X Cao, X Xie, J Wang and W Wang. Single-base resolution maps of cultivated and wild rice methylomes and regulatory roles of DNA methylation in plant gene expression. BMC Genomics 13, 300. (2012)</a></p></li>
<li><p><a href="https://doi.org/10.1371/journal.pone.0040637">216.	X Jin, M He, B Ferguson, Y Meng, L Ouyang, J Ren, T Mailund, F Sun, L Sun, J Shen, M Zhuo, L Song, J Wang, F Ling, Y Zhu, C Hvilsom, H Siegismund, X Liu, Z Gong, F Ji, X Wang, B Liu, Y Zhang, J Hou, J Wang, H Zhao, Y Wang, X Fang, <b>G Zhang</b>, J Wang, X Zhang, MH Schierup, H Du, J Wang and X Wang. An Effort to Use Human-Based Exome Capture Methods to Analyze Chimpanzee and Macaque Exomes. PLOS ONE 7, e40637. (2012)</a></p></li>
<li><p><a href="https://doi.org/10.1038/nature11622">MaM Groenen, AL Archibald, H Uenishi, CK Tuggle, Y Takeuchi, MF Rothschild, C Rogel-Gaillard, C Park, D Milan, H-J Megens, S Li, DM Larkin, H Kim, LaF Frantz, M Caccamo, H Ahn, BL Aken, A Anselmo, C Anthon, L Auvil, B Badaoui, CW Beattie, C Bendixen, D Berman, F Blecha, J Blomberg, L Bolund, M Bosse, S Botti, Z Bujie, M Bystrom, B Capitanu, D Carvalho-Silva, P Chardon, C Chen, R Cheng, S-H Choi, W Chow, RC Clark, C Clee, RPMA Crooijmans, HD Dawson, P Dehais, F De Sapio, B Dibbits, N Drou, Z-Q Du, K Eversole, J Fadista, S Fairley, T Faraut, GJ Faulkner, KE Fowler, M Fredholm, E Fritz, JGR Gilbert, E Giuffra, J Gorodkin, DK Griffin, JL Harrow, A Hayward, K Howe, Z-L Hu, SJ Humphray, T Hunt, H Hornshoj, J-T Jeon, P Jern, M Jones, J Jurka, H Kanamori, R Kapetanovic, J Kim, J-H Kim, K-W Kim, T-H Kim, G Larson, K Lee, K-T Lee, R Leggett, HA Lewin, Y Li, W Liu, JE Loveland, Y Lu, JK Lunney, J Ma, O Madsen, K Mann, L Matthews, S Mclaren, T Morozumi, MP Murtaugh, J Narayan, D Truong Nguyen, P Ni, S-J Oh, S Onteru, F Panitz, E-W Park, H-S Park, G Pascal, Y Paudel, M Perez-Enciso, R Ramirez-Gonzalez, JM Reecy, S Rodriguez-Zas, GA Rohrer, L Rund, Y Sang, K Schachtschneider, JG Schraiber, J Schwartz, L Scobie, C Scott, S Searle, B Servin, BR Southey, G Sperber, P Stadler, JV Sweedler, H Tafer, B Thomsen, R Wali, J Wang, J Wang, S White, X Xu, M Yerle, <b>G Zhang</b>, J Zhang, J Zhang, S Zhao, J Rogers, C Churcher and LB Schook. Analyses of pig genomes provide insight into porcine demography and evolution. Nature 491, 393-398. (2012)</a></p></li>
<li><p><a href="https://doi.org/10.1186/2047-217X-1-16">X Fang, Y Mou, Z Huang, Y Li, L Han, Y Zhang, Y Feng, Y Chen, X Jiang, W Zhao, X Sun, Z Xiong, L Yang, H Liu, D Fan, L Mao, L Ren, C Liu, J Wang, K Li, G Wang, S Yang, L Lai, <b>G Zhang</b>, Y Li, J Wang, L Bolund, H Yang, J Wang, S Feng, S Li and Y Du. The sequence and analysis of a Chinese pig genome. GigaScience 1, 16. (2012)</a></p></li>
<li><p><a href="https://doi.org/10.1371/journal.pone.0031645">F Chen, W Zhang, Y Liang, J Huang, K Li, CD Green, J Liu, <b>G Zhang</b>, B Zhou, X Yi, W Wang, H Liu, X Xu, F Shen, N Qu, Y Wang, G Gao, A San, L Jiangbai, H Sang, X Fang, K Kristiansen, H Yang, J Wang, J-DJ Han and J Wang. Transcriptome and Network Changes in Climbers at Extreme Altitudes. PLOS ONE 7, e31645. (2012)</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.cub.2012.07.042">R Bonasio, Q Li, J Lian, Navdeep s Mutti, L Jin, H Zhao, P Zhang, P Wen, H Xiang, Y Ding, Z Jin, Steven s Shen, Z Wang, W Wang, J Wang, Shelley l Berger, J Liebig, <b>G Zhang</b> and D Reinberg. Genome-wide and Caste-Specific DNA Methylomes of the Ants Camponotus floridanus and Harpegnathos saltator. Current Biology 22, 1755-1764. (2012)</a></p></li>
<li><p><a href="https://doi.org/10.1098/rspb.2012.1745">ME Allentoft, M Collins, D Harker, J Haile, CL Oskam, ML Hale, PF Campos, JA Samaniego, MTP Gilbert, E Willerslev, <b>G Zhang</b>, RP Scofield, RN Holdaway and M Bunce. The half-life of DNA in bone: measuring decay kinetics in 158 dated fossils. Proceedings of the Royal Society B: Biological Sciences 279, 4724-4733. (2012)</a></p></li>
<li><p><a href="https://doi.org/10.4056/sigs.3136559">CB Albertin, L Bonnaud, CT Brown, WJ Crookes-Goodson, RR Da Fonseca, C Di Cristo, BP Dilkes, E Edsinger-Gonzales, RM Freeman, RT Hanlon, KM Koenig, AR Lindgren, MQ Martindale, P Minx, LL Moroz, M-T Nodl, SV Nyholm, A Ogura, JR Pungor, JJC Rosenthal, EM Schwarz, S Shigeno, JM Strugnell, T Wollesen, <b>G Zhang</b> and CW Ragsdale. Cephalopod genomics: A plan of strategies and organization. Standards in Genomic Sciences 7, 175-188. (2012)</a></p></li>

<h2>-- 2011年研究成果 --</h2>
<li><p><a href="https://doi.org/10.1186/1479-7364-5-5-453"><b>G Zhang</b>, Z Pei, EV Ball, M Mort, H Kehrer-Sawatzki and DN Cooper. Cross-comparison of the genome sequences from human, chimpanzee, Neanderthal and a Denisovan hominin identifies novel potentially compensated mutations. Human Genomics 5, 453. (2011)</a></p></li>
<li><p><a href="https://doi.org/10.1038/nbt.1992">G Yan, <b>G Zhang</b>, X Fang, Y Zhang, C Li, F Ling, DN Cooper, Q Li, Y Li, AJ Van Gool, H Du, J Chen, R Chen, P Zhang, Z Huang, JR Thompson, Y Meng, Y Bai, J Wang, M Zhuo, T Wang, Y Huang, L Wei, J Li, Z Wang, H Hu, P Yang, L Le, PD Stenson, B Li, X Liu, EV Ball, N An, Q Huang, Y Zhang, W Fan, X Zhang, Y Li, W Wang, MG Katze, B Su, R Nielsen, H Yang, J Wang, X Wang and J Wang. Genome sequencing and comparison of two nonhuman primate animal models, the cynomolgus and Chinese rhesus macaques. Nature Biotechnology 29, 1019-1023. (2011)</a></p></li>
<li><p><a href="https://genome.cshlp.org/content/21/8/1339">S Nygaard, <b>G Zhang</b>, M Schiott, C Li, Y Wurm, H Hu, J Zhou, L Ji, F Qiu, M Rasmussen, H Pan, F Hauser, A Krogh, CJ Grimmelikhuijzen, J Wang and JJ Boomsma. The genome of the leaf-cutting ant Acromyrmex echinatior suggests key adaptations to advanced social life and fungus farming. Genome research 21, 1339-1348. (2011)</a></p></li>
<li><p><a href="https://academic.oup.com/gigascience/search-results?page=1&q=Genomic+data+from+the+polar+bear+(Ursus+maritimus)&fl_SiteID=5589&SearchSourceType=1&allJournals=1&login=false">B Li, <b>G Zhang</b>, E Willerslev, J Wang and J Wang. Genomic data from the polar bear (Ursus maritimus). GigaScience 10, 10-5524. (2011)</a></p></li>
<li><p><a href="https://doi.org/10.1038/nature10533">EB Kim, X Fang, AA Fushan, Z Huang, AV Lobanov, L Han, SM Marino, X Sun, AA Turanov, P Yang, SH Yim, X Zhao, MV Kasaikina, N Stoletzki, C Peng, P Polak, Z Xiong, A Kiezun, Y Zhu, Y Chen, GV Kryukov, Q Zhang, L Peshkin, L Yang, RT Bronson, R Buffenstein, B Wang, C Han, Q Li, L Chen, W Zhao, SR Sunyaev, TJ Park, <b>G Zhang</b>, J Wang and VN Gladyshev. Genome sequencing reveals insights into physiology and longevity of the naked mole rat. Nature 479, 223-227. (2011)</a></p></li>
<li><p><a href="https://doi.org/10.1038/nature10553">AR Jex, S Liu, B Li, ND Young, RS Hall, Y Li, L Yang, N Zeng, X Xu, Z Xiong, F Chen, X Wu, <b>G Zhang</b>, X Fang, Y Kang, GA Anderson, TW Harris, BE Campbell, J Vlaminck, T Wang, C Cantacessi, EM Schwarz, S Ranganathan, P Geldhof, P Nejsum, PW Sternberg, H Yang, J Wang, J Wang and RB Gasser. Ascaris suum draft genome. Nature 479, 529-533. (2011)</a></p></li>
<li><p><a href="https://pubmed.ncbi.nlm.nih.gov/22199030/">VN Gladyshev, <b>G Zhang</b> and J Wang. The naked mole rat genome: understanding aging through genome analysis. Aging13, 12. (2011)</a></p></li>

<h2>-- 2010年研究成果 --</h2>
<li><p><a href="https://doi.org/10.1002/humu.21389"><b>G Zhang</b>, Z Pei, M Krawczak, EV Ball, M Mort, H Kehrer-Sawatzki and DN Cooper. Triangulation of the human, chimpanzee, and Neanderthal genome sequences identifies potentially compensated mutations. Human Mutation 31, 1286-1293. (2010)</a></p></li>
<li><p><a href="https://genome.cshlp.org/content/20/5/646"><b>G Zhang</b>, G Guo, X Hu, Y Zhang, Q Li, R Li, R Zhuang, Z Lu, Z He, X Fang, L Chen, W Tian, Y Tao, K Kristiansen, X Zhang, S Li, H Yang, J Wang and J Wang. Deep RNA sequencing at single base-pair resolution reveals high complexity of the rice transcriptome. Genome research 20, 646-654. (2010)</a></p></li>
<li><p><a href="https://doi.org/10.1038/nbt.1626">H Xiang, J Zhu, Q Chen, F Dai, X Li, M Li, H Zhang, <b>G Zhang</b>, D Li, Y Dong, L Zhao, Y Lin, D Cheng, J Yu, J Sun, X Zhou, K Ma, Y He, Y Zhao, S Guo, M Ye, G Guo, Y Li, R Li, X Zhang, L Ma, K Kristiansen, Q Guo, J Jiang, S Beck, Q Xia, W Wang and J Wang. Single base-resolution methylome of the silkworm reveals a sparse epigenomic map. Nature Biotechnology 28, 516-520. (2010)</a></p></li>
<li><p><a href="https://doi.org/10.1038/nature08696">R Li, W Fan, G Tian, H Zhu, L He, J Cai, Q Huang, Q Cai, B Li, Y Bai, Z Zhang, Y Zhang, W Wang, J Li, F Wei, H Li, M Jian, J Li, Z Zhang, R Nielsen, D Li, W Gu, Z Yang, Z Xuan, OA Ryder, FC-C Leung, Y Zhou, J Cao, X Sun, Y Fu, X Fang, X Guo, B Wang, R Hou, F Shen, B Mu, P Ni, R Lin, W Qian, G Wang, C Yu, W Nie, J Wang, Z Wu, H Liang, J Min, Q Wu, S Cheng, J Ruan, M Wang, Z Shi, M Wen, B Liu, X Ren, H Zheng, D Dong, K Cook, G Shan, H Zhang, C Kosiol, X Xie, Z Lu, H Zheng, Y Li, CC Steiner, TT-Y Lam, S Lin, Q Zhang, G Li, J Tian, T Gong, H Liu, D Zhang, L Fang, C Ye, J Zhang, W Hu, A Xu, Y Ren, <b>G Zhang</b>, MW Bruford, Q Li, L Ma, Y Guo, N An, Y Hu, Y Zheng, Y Shi, Z Li, Q Liu, Y Chen, J Zhao, N Qu, S Zhao, F Tian, X Wang, H Wang, L Xu, X Liu, T Vinar, Y Wang, T-W Lam, S-M Yiu, S Liu, H Zhang, D Li, Y Huang, X Wang, G Yang, Z Jiang, J Wang, N Qin, L Li, J Li, L Bolund, K Kristiansen, GK-S Wong, M Olson, X Zhang, S Li, H Yang, J Wang and J Wang. The sequence and de novo assembly of the giant panda genome. Nature 463, 311-317. (2010)</a></p></li>
<li><p><a href="https://doi.org/10.1371/journal.pgen.1001255">Y Ding, L Zhao, S Yang, Y Jiang, Y Chen, R Zhao, Y Zhang, <b>G Zhang</b>, Y Dong, H Yu, Q Zhou and W Wang. A Young Drosophila Duplicate Gene Plays Essential Roles in Spermatogenesis by Regulating Several Y-Linked Male Fertility Genes. PLOS Genetics 6, e1001255. (2010)</a></p></li>
<li><p><a href="https://doi.org/10.1126/science.1192428">R Bonasio, <b>G Zhang</b>, C Ye, NS Mutti, X Fang, N Qin, G Donahue, P Yang, Q Li, C Li, P Zhang, Z Huang, SL Berger, D Reinberg, J Wang and J Liebig. Genomic Comparison of the Ants Camponotus floridanus and Harpegnathos saltator. Science 329, 1068-1071. (2010)</a></p></li>

<h2>-- 早于2010年的研究成果 --</h2>
<li><p><a href="https://doi.org/10.1126/science.1176620">Q Xia, Y Guo, Z Zhang, D Li, Z Xuan, Z Li, F Dai, Y Li, D Cheng, R Li, T Cheng, T Jiang, C Becquet, X Xu, C Liu, X Zha, W Fan, Y Lin, Y Shen, L Jiang, J Jensen, I Hellmann, S Tang, P Zhao, H Xu, C Yu, <b>G Zhang</b>, J Li, J Cao, S Liu, N He, Y Zhou, H Liu, J Zhao, C Ye, Z Du, G Pan, A Zhao, H Shao, W Zeng, P Wu, C Li, M Pan, J Li, X Yin, D Li, J Wang, H Zheng, W Wang, X Zhang, S Li, H Yang, C Lu, R Nielsen, Z Zhou, J Wang, Z Xiang and J Wang. Complete Resequencing of 40 Genomes Reveals Domestication Events and Genes in Silkworm (Bombyx). Science 326, 433-436. (2009)</a></p></li>
<li><p><a href="https://doi.org/10.1038/ng.475">S Huang, R Li, Z Zhang, L Li, X Gu, W Fan, WJ Lucas, X Wang, B Xie, P Ni, Y Ren, H Zhu, J Li, K Lin, W Jin, Z Fei, G Li, J Staub, A Kilian, EaG Van Der Vossen, Y Wu, J Guo, J He, Z Jia, Y Ren, G Tian, Y Lu, J Ruan, W Qian, M Wang, Q Huang, B Li, Z Xuan, J Cao, Asan, Z Wu, J Zhang, Q Cai, Y Bai, B Zhao, Y Han, Y Li, X Li, S Wang, Q Shi, S Liu, WK Cho, J-Y Kim, Y Xu, K Heller-Uszynska, H Miao, Z Cheng, S Zhang, J Wu, Y Yang, H Kang, M Li, H Liang, X Ren, Z Shi, M Wen, M Jian, H Yang, <b>G Zhang</b>, Z Yang, R Chen, S Liu, J Li, L Ma, H Liu, Y Zhou, J Zhao, X Fang, G Li, L Fang, Y Li, D Liu, H Zheng, Y Zhang, N Qin, Z Li, G Yang, S Yang, L Bolund, K Kristiansen, H Zheng, S Li, X Zhang, H Yang, J Wang, R Sun, B Zhang, S Jiang, J Wang, Y Du and S Li. The genome of the cucumber, Cucumis sativus L. Nature Genetics 41, 1275-1281. (2009)</a></p></li>
<li><p><a href="https://genome.cshlp.org/content/18/9/1446">Q Zhou, <b>G Zhang</b>, Y Zhang, S Xu, R Zhao, Z Zhan, X Li, Y Ding, S Yang and W Wang. On the origin of new genes in Drosophila. Genome research 18, 1446-1455. (2008)</a></p></li>
<li><p><a href="https://doi.org/10.1038/nature07484">J Wang, W Wang, R Li, Y Li, G Tian, L Goodman, W Fan, J Zhang, J Li, J Zhang, Y Guo, B Feng, H Li, Y Lu, X Fang, H Liang, Z Du, D Li, Y Zhao, Y Hu, Z Yang, H Zheng, I Hellmann, M Inouye, J Pool, X Yi, J Zhao, J Duan, Y Zhou, J Qin, L Ma, G Li, Z Yang, <b>G Zhang</b>, B Yang, C Yu, F Liang, W Li, S Li, D Li, P Ni, J Ruan, Q Li, H Zhu, D Liu, Z Lu, N Li, G Guo, J Zhang, J Ye, L Fang, Q Hao, Q Chen, Y Liang, Y Su, A San, C Ping, S Yang, F Chen, L Li, K Zhou, H Zheng, Y Ren, L Yang, Y Gao, G Yang, Z Li, X Feng, K Kristiansen, GK-S Wong, R Nielsen, R Durbin, L Bolund, X Zhang, S Li, H Yang and J Wang. The diploid genome sequence of an Asian individual. Nature 456, 60-65. (2008)</a></p></li>
<li><p><a href="https://doi.org/10.1186/1471-2164-8-93"><b>G Zhang</b>, H Wang, J Shi, X Wang, H Zheng, GK-S Wong, T Clark, W Wang, J Wang and L Kang. Identification and characterization of insect-specific proteins by genome data analysis. BMC Genomics 8, 93. (2007)</a></p></li>
<li><p><a href="https://doi.org/10.1105/tpc.106.041905">W Wang, H Zheng, C Fan, J Li, J Shi, Z Cai, <b>G Zhang</b>, D Liu, J Zhang, S Vang, Z Lu, GK-S Wong, M Long and J Wang. High Rate of Chimeric Gene Origination by Retroposition in Plant Genomes. The Plant Cell 18, 1791-1802. (2006)</a></p></li>
                                </ul>
                        -->
                        </div>
                </div>
				<!--
                <div class="f_bottom1" style="top:23900px"><div class="bottom1">© Copyright 2019 by Centre for Biodiversity Genomics, All Rights Reserved</div></div>
				-->
                <div class="f_bottom1" style="top:2300px"><div class="bottom1">©华中师范大学生命科学学院焦恒武实验室&nbsp; 地址:武汉市珞喻路152号生命科学学院&nbsp;&nbsp; 邮政编码:430072</div></div>
				
        </body>
</html>
